2023-02-15 00:55:41.390 [info] <0.5783.0>@vmq_swc_peer_service:attempt_join:{51,5} Sent join request to: 'VerneMQ1@127.0.0.2'
2023-02-15 00:55:48.340 [info] <0.5783.0>@vmq_swc_peer_service:attempt_join:{54,13} Unable to connect to 'VerneMQ1@127.0.0.2'
2023-02-15 00:55:57.254 [info] <0.5795.0>@vmq_swc_peer_service:attempt_join:{51,5} Sent join request to: 'VerneMQ1@127.0.0.2'
2023-02-15 00:56:04.192 [info] <0.5795.0>@vmq_swc_peer_service:attempt_join:{54,13} Unable to connect to 'VerneMQ1@127.0.0.2'
2023-02-15 01:02:38.154 [info] <0.89.0> alarm_handler: {set,{system_memory_high_watermark,[]}}
2023-02-15 01:02:38.215 [info] <0.222.0>@vmq_swc_peer_service_manager:write_old_actor_to_disk:{180,13} writing (updated) old actor <<239,174,60,145,53,39,215,115,159,70,176,124,80,197,199,248,97,198,2,60>> to disk
2023-02-15 01:02:38.218 [info] <0.222.0>@vmq_swc_peer_service_manager:write_state_to_disk:{163,13} writing state {[{[{actor,<<239,174,60,145,53,39,215,115,159,70,176,124,80,197,199,248,97,198,2,60>>}],1}],{dict,1,16,16,8,80,48,{[],[],[],[],[],[],[],[],[],[],[],[],[],[],[],[]},{{[],[['VerneMQ@127.0.0.1',{[{actor,<<239,174,60,145,53,39,215,115,159,70,176,124,80,197,199,248,97,198,2,60>>}],1}]],[],[],[],[],[],[],[],[],[],[],[],[],[],[]}}},{dict,0,16,16,8,80,48,{[],[],[],[],[],[],[],[],[],[],[],[],[],[],[],[]},{{[],[],[],[],[],[],[],[],[],[],[],[],[],[],[],[]}}}} to disk <<75,2,131,80,0,0,0,247,120,1,203,96,206,97,96,96,96,204,96,130,82,41,12,172,137,201,37,249,69,185,64,81,145,247,235,108,38,154,170,95,47,158,239,182,161,38,224,232,241,31,137,199,152,108,178,18,25,179,50,56,83,24,88,82,50,147,75,18,25,19,5,128,144,35,49,32,209,32,67,32,11,13,100,48,2,197,192,230,130,8,166,20,6,193,176,212,162,188,84,223,64,7,67,35,115,61,3,32,52,36,205,94,116,243,225,206,96,32,232,12,52,173,0,211,246,78,109>>
2023-02-15 01:02:38.239 [info] <0.228.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta1" in 2000 ms after error IO error: lock ./data/swc_meta/meta1/LOCK: Resource temporarily unavailable
2023-02-15 01:02:40.240 [info] <0.228.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta1" in 2000 ms after error IO error: lock ./data/swc_meta/meta1/LOCK: Resource temporarily unavailable
2023-02-15 01:02:42.244 [info] <0.228.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta1" in 2000 ms after error IO error: lock ./data/swc_meta/meta1/LOCK: Resource temporarily unavailable
2023-02-15 01:02:44.247 [info] <0.228.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta1" in 2000 ms after error IO error: lock ./data/swc_meta/meta1/LOCK: Resource temporarily unavailable
2023-02-15 01:02:46.250 [info] <0.228.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta1" in 2000 ms after error IO error: lock ./data/swc_meta/meta1/LOCK: Resource temporarily unavailable
2023-02-15 01:02:48.252 [info] <0.228.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta1" in 2000 ms after error IO error: lock ./data/swc_meta/meta1/LOCK: Resource temporarily unavailable
2023-02-15 01:02:50.256 [info] <0.228.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta1" in 2000 ms after error IO error: lock ./data/swc_meta/meta1/LOCK: Resource temporarily unavailable
2023-02-15 01:02:52.260 [info] <0.228.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta1" in 2000 ms after error IO error: lock ./data/swc_meta/meta1/LOCK: Resource temporarily unavailable
2023-02-15 01:02:53.241 [info] <0.437.0>@vmq_cluster_mon:handle_info:{129,5} cluster node 'VerneMQ1@127.0.0.2' UP
2023-02-15 01:02:54.261 [info] <0.228.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta1" in 2000 ms after error IO error: lock ./data/swc_meta/meta1/LOCK: Resource temporarily unavailable
2023-02-15 01:02:56.266 [info] <0.228.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta1" in 2000 ms after error IO error: lock ./data/swc_meta/meta1/LOCK: Resource temporarily unavailable
2023-02-15 01:02:58.269 [info] <0.228.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta1" in 2000 ms after error IO error: lock ./data/swc_meta/meta1/LOCK: Resource temporarily unavailable
2023-02-15 01:03:00.273 [info] <0.228.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta1" in 2000 ms after error IO error: lock ./data/swc_meta/meta1/LOCK: Resource temporarily unavailable
2023-02-15 01:03:02.276 [info] <0.228.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta1" in 2000 ms after error IO error: lock ./data/swc_meta/meta1/LOCK: Resource temporarily unavailable
2023-02-15 01:03:04.213 [info] <0.5941.0> Administrative stop
2023-02-15 01:03:04.240 [info] <0.90.0> alarm_handler: {clear,system_memory_high_watermark}
2023-02-15 01:03:04.282 [info] <0.228.0>@vmq_swc_db_leveldb:open_db:{269,13} Opening LevelDB SWC database at "./data/swc_meta/meta1"
2023-02-15 01:03:04.294 [info] <0.245.0>@vmq_swc_db_leveldb:open_db:{269,13} Opening LevelDB SWC database at "./data/swc_meta/meta2"
2023-02-15 01:03:04.297 [info] <0.254.0>@vmq_swc_db_leveldb:open_db:{269,13} Opening LevelDB SWC database at "./data/swc_meta/meta3"
2023-02-15 01:03:04.301 [info] <0.263.0>@vmq_swc_db_leveldb:open_db:{269,13} Opening LevelDB SWC database at "./data/swc_meta/meta4"
2023-02-15 01:03:04.305 [info] <0.272.0>@vmq_swc_db_leveldb:open_db:{269,13} Opening LevelDB SWC database at "./data/swc_meta/meta5"
2023-02-15 01:03:04.308 [info] <0.281.0>@vmq_swc_db_leveldb:open_db:{269,13} Opening LevelDB SWC database at "./data/swc_meta/meta6"
2023-02-15 01:03:04.312 [info] <0.290.0>@vmq_swc_db_leveldb:open_db:{269,13} Opening LevelDB SWC database at "./data/swc_meta/meta7"
2023-02-15 01:03:04.317 [info] <0.299.0>@vmq_swc_db_leveldb:open_db:{269,13} Opening LevelDB SWC database at "./data/swc_meta/meta8"
2023-02-15 01:03:04.321 [info] <0.308.0>@vmq_swc_db_leveldb:open_db:{269,13} Opening LevelDB SWC database at "./data/swc_meta/meta9"
2023-02-15 01:03:04.325 [info] <0.317.0>@vmq_swc_db_leveldb:open_db:{269,13} Opening LevelDB SWC database at "./data/swc_meta/meta10"
2023-02-15 01:03:04.534 [info] <0.215.0>@vmq_metadata:start:{29,5} Try to start vmq_swc: ok
2023-02-15 01:03:04.556 [info] <0.330.0>@vmq_storage_engine_leveldb:open_db:{145,13} Opening LevelDB database at "./data/msgstore/1"
2023-02-15 01:03:04.557 [info] <0.331.0>@vmq_storage_engine_leveldb:open_db:{145,13} Opening LevelDB database at "./data/msgstore/2"
2023-02-15 01:03:04.559 [info] <0.332.0>@vmq_storage_engine_leveldb:open_db:{145,13} Opening LevelDB database at "./data/msgstore/3"
2023-02-15 01:03:04.561 [info] <0.333.0>@vmq_storage_engine_leveldb:open_db:{145,13} Opening LevelDB database at "./data/msgstore/4"
2023-02-15 01:03:04.562 [info] <0.334.0>@vmq_storage_engine_leveldb:open_db:{145,13} Opening LevelDB database at "./data/msgstore/5"
2023-02-15 01:03:04.563 [info] <0.335.0>@vmq_storage_engine_leveldb:open_db:{145,13} Opening LevelDB database at "./data/msgstore/6"
2023-02-15 01:03:04.565 [info] <0.336.0>@vmq_storage_engine_leveldb:open_db:{145,13} Opening LevelDB database at "./data/msgstore/7"
2023-02-15 01:03:04.566 [info] <0.337.0>@vmq_storage_engine_leveldb:open_db:{145,13} Opening LevelDB database at "./data/msgstore/8"
2023-02-15 01:03:04.569 [info] <0.338.0>@vmq_storage_engine_leveldb:open_db:{145,13} Opening LevelDB database at "./data/msgstore/9"
2023-02-15 01:03:04.570 [info] <0.339.0>@vmq_storage_engine_leveldb:open_db:{145,13} Opening LevelDB database at "./data/msgstore/10"
2023-02-15 01:03:04.571 [info] <0.340.0>@vmq_storage_engine_leveldb:open_db:{145,13} Opening LevelDB database at "./data/msgstore/11"
2023-02-15 01:03:04.573 [info] <0.341.0>@vmq_storage_engine_leveldb:open_db:{145,13} Opening LevelDB database at "./data/msgstore/12"
2023-02-15 01:03:04.592 [info] <0.215.0>@vmq_message_store:start:{29,5} Try to start vmq_generic_msg_store: ok
2023-02-15 01:03:04.595 [info] <0.343.0>@vmq_server_sup:maybe_change_nodename:{62,13} rename VerneMQ node from 'VerneMQ@127.0.0.1' to 'VerneMQ1@127.0.0.2'
2023-02-15 01:03:04.596 [info] <0.225.0>@vmq_swc_peer_service_manager:write_state_to_disk:{163,13} writing state {[{[{actor,<<239,174,60,145,53,39,215,115,159,70,176,124,80,197,199,248,97,198,2,60>>}],1},{<<239,174,60,145,53,39,215,115,159,70,176,124,80,197,199,248,97,198,2,60>>,1}],{dict,1,16,16,8,80,48,{[],[],[],[],[],[],[],[],[],[],[],[],[],[],[],[]},{{[],[],[['VerneMQ1@127.0.0.2',{<<239,174,60,145,53,39,215,115,159,70,176,124,80,197,199,248,97,198,2,60>>,1}]],[],[],[],[],[],[],[],[],[],[],[],[],[]}}},{dict,0,16,16,8,80,48,{[],[],[],[],[],[],[],[],[],[],[],[],[],[],[],[]},{{[],[],[],[],[],[],[],[],[],[],[],[],[],[],[],[]}}}} to disk <<75,2,131,80,0,0,1,5,120,1,203,96,206,97,96,96,96,202,96,2,81,140,25,76,41,12,172,137,201,37,249,69,185,64,174,200,251,117,54,19,77,213,175,23,207,119,219,80,19,112,244,248,143,196,99,76,54,89,137,64,85,56,101,19,25,179,50,56,83,24,88,82,50,147,75,18,25,19,5,128,144,35,49,32,209,32,67,32,11,13,100,48,130,196,192,182,130,93,144,194,32,20,150,90,148,151,234,27,104,232,96,104,100,174,103,0,132,70,248,109,66,53,17,97,47,3,97,123,81,181,2,0,216,251,86,133>>
2023-02-15 01:03:04.661 [info] <0.451.0>@vmq_reg_trie:handle_info:{254,5} loaded 0 subscriptions into vmq_reg_trie
2023-02-15 01:03:04.663 [info] <0.226.0>@vmq_cluster:init:{163,5} cluster event handler 'vmq_cluster' registered
2023-02-15 01:03:04.680 [error] <0.467.0>@vmq_ranch_config:reconfigure_listeners_for_type:{269,13} can't reconfigure mqtts listener({127,0,0,1}, 1883) with Options [{max_connections,10000},{nr_of_acceptors,10},{mountpoint,[]},{depth,1},{eccs,[sect571r1,sect571k1,secp521r1,brainpoolP512r1,sect409k1,sect409r1,brainpoolP384r1,secp384r1,sect283k1,sect283r1,brainpoolP256r1,secp256k1,secp256r1,sect239k1,sect233k1,sect233r1,secp224k1,secp224r1,sect193r1,sect193r2,secp192k1,secp192r1,sect163k1,sect163r1,sect163r2,secp160k1,secp160r1,secp160r2]},{require_certificate,false},{tls_version,'tlsv1.2'},{use_identity_as_username,false},{allowed_protocol_versions,[3,4,131]},{allow_anonymous_override,false}] due to {already_started,<0.468.0>}
2023-02-15 01:03:05.250 [warning] <0.464.0>@vmq_cluster_mon:handle_info:{125,5} cluster node 'VerneMQ@127.0.0.1' DOWN
2023-02-15 01:03:14.967 [info] <0.89.0> alarm_handler: {set,{system_memory_high_watermark,[]}}
2023-02-15 01:03:15.025 [info] <0.222.0>@vmq_swc_peer_service_manager:write_old_actor_to_disk:{180,13} writing (updated) old actor <<239,174,60,145,53,39,215,115,159,70,176,124,80,197,199,248,97,198,2,60>> to disk
2023-02-15 01:03:15.029 [info] <0.222.0>@vmq_swc_peer_service_manager:write_state_to_disk:{163,13} writing state {[{[{actor,<<239,174,60,145,53,39,215,115,159,70,176,124,80,197,199,248,97,198,2,60>>}],1},{<<239,174,60,145,53,39,215,115,159,70,176,124,80,197,199,248,97,198,2,60>>,1}],{dict,1,16,16,8,80,48,{[],[],[],[],[],[],[],[],[],[],[],[],[],[],[],[]},{{[],[],[['VerneMQ1@127.0.0.2',{<<239,174,60,145,53,39,215,115,159,70,176,124,80,197,199,248,97,198,2,60>>,1}]],[],[],[],[],[],[],[],[],[],[],[],[],[]}}},{dict,0,16,16,8,80,48,{[],[],[],[],[],[],[],[],[],[],[],[],[],[],[],[]},{{[],[],[],[],[],[],[],[],[],[],[],[],[],[],[],[]}}}} to disk <<75,2,131,80,0,0,1,5,120,1,203,96,206,97,96,96,96,202,96,2,81,140,25,76,41,12,172,137,201,37,249,69,185,64,174,200,251,117,54,19,77,213,175,23,207,119,219,80,19,112,244,248,143,196,99,76,54,89,137,64,85,56,101,19,25,179,50,56,83,24,88,82,50,147,75,18,25,19,5,128,144,35,49,32,209,32,67,32,11,13,100,48,130,196,192,182,130,93,144,194,32,20,150,90,148,151,234,27,104,232,96,104,100,174,103,0,132,70,248,109,66,53,17,97,47,3,97,123,81,181,2,0,216,251,86,133>>
2023-02-15 01:03:15.045 [info] <0.228.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta1" in 2000 ms after error IO error: lock ./data/swc_meta/meta1/LOCK: Resource temporarily unavailable
2023-02-15 01:03:17.048 [info] <0.228.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta1" in 2000 ms after error IO error: lock ./data/swc_meta/meta1/LOCK: Resource temporarily unavailable
2023-02-15 01:03:19.049 [info] <0.228.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta1" in 2000 ms after error IO error: lock ./data/swc_meta/meta1/LOCK: Resource temporarily unavailable
2023-02-15 01:03:21.052 [info] <0.228.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta1" in 2000 ms after error IO error: lock ./data/swc_meta/meta1/LOCK: Resource temporarily unavailable
2023-02-15 01:03:23.055 [info] <0.228.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta1" in 2000 ms after error IO error: lock ./data/swc_meta/meta1/LOCK: Resource temporarily unavailable
2023-02-15 01:03:25.057 [info] <0.228.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta1" in 2000 ms after error IO error: lock ./data/swc_meta/meta1/LOCK: Resource temporarily unavailable
2023-02-15 01:03:27.060 [info] <0.228.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta1" in 2000 ms after error IO error: lock ./data/swc_meta/meta1/LOCK: Resource temporarily unavailable
2023-02-15 01:03:29.061 [info] <0.228.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta1" in 2000 ms after error IO error: lock ./data/swc_meta/meta1/LOCK: Resource temporarily unavailable
2023-02-15 01:03:31.064 [info] <0.228.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta1" in 2000 ms after error IO error: lock ./data/swc_meta/meta1/LOCK: Resource temporarily unavailable
2023-02-15 01:03:33.068 [info] <0.228.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta1" in 2000 ms after error IO error: lock ./data/swc_meta/meta1/LOCK: Resource temporarily unavailable
2023-02-15 01:03:35.070 [info] <0.228.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta1" in 2000 ms after error IO error: lock ./data/swc_meta/meta1/LOCK: Resource temporarily unavailable
2023-02-15 01:03:37.073 [info] <0.228.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta1" in 2000 ms after error IO error: lock ./data/swc_meta/meta1/LOCK: Resource temporarily unavailable
2023-02-15 01:03:39.076 [info] <0.228.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta1" in 2000 ms after error IO error: lock ./data/swc_meta/meta1/LOCK: Resource temporarily unavailable
2023-02-15 01:03:41.079 [info] <0.228.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta1" in 2000 ms after error IO error: lock ./data/swc_meta/meta1/LOCK: Resource temporarily unavailable
2023-02-15 01:03:43.083 [info] <0.228.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta1" in 2000 ms after error IO error: lock ./data/swc_meta/meta1/LOCK: Resource temporarily unavailable
2023-02-15 01:03:45.087 [info] <0.228.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta1" in 2000 ms after error IO error: lock ./data/swc_meta/meta1/LOCK: Resource temporarily unavailable
2023-02-15 01:03:47.091 [info] <0.228.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta1" in 2000 ms after error IO error: lock ./data/swc_meta/meta1/LOCK: Resource temporarily unavailable
2023-02-15 01:03:49.094 [info] <0.228.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta1" in 2000 ms after error IO error: lock ./data/swc_meta/meta1/LOCK: Resource temporarily unavailable
2023-02-15 01:03:51.098 [info] <0.228.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta1" in 2000 ms after error IO error: lock ./data/swc_meta/meta1/LOCK: Resource temporarily unavailable
2023-02-15 01:03:53.101 [info] <0.228.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta1" in 2000 ms after error IO error: lock ./data/swc_meta/meta1/LOCK: Resource temporarily unavailable
2023-02-15 01:03:55.105 [info] <0.228.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta1" in 2000 ms after error IO error: lock ./data/swc_meta/meta1/LOCK: Resource temporarily unavailable
2023-02-15 01:03:57.108 [info] <0.228.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta1" in 2000 ms after error IO error: lock ./data/swc_meta/meta1/LOCK: Resource temporarily unavailable
2023-02-15 01:03:59.109 [info] <0.228.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta1" in 2000 ms after error IO error: lock ./data/swc_meta/meta1/LOCK: Resource temporarily unavailable
2023-02-15 01:04:01.111 [info] <0.228.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta1" in 2000 ms after error IO error: lock ./data/swc_meta/meta1/LOCK: Resource temporarily unavailable
2023-02-15 01:04:03.115 [info] <0.228.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta1" in 2000 ms after error IO error: lock ./data/swc_meta/meta1/LOCK: Resource temporarily unavailable
2023-02-15 01:04:05.118 [info] <0.228.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta1" in 2000 ms after error IO error: lock ./data/swc_meta/meta1/LOCK: Resource temporarily unavailable
2023-02-15 01:04:07.128 [info] <0.228.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta1" in 2000 ms after error IO error: lock ./data/swc_meta/meta1/LOCK: Resource temporarily unavailable
2023-02-15 01:04:09.131 [info] <0.228.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta1" in 2000 ms after error IO error: lock ./data/swc_meta/meta1/LOCK: Resource temporarily unavailable
2023-02-15 01:04:11.134 [info] <0.228.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta1" in 2000 ms after error IO error: lock ./data/swc_meta/meta1/LOCK: Resource temporarily unavailable
2023-02-15 01:04:13.136 [info] <0.228.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta1" in 2000 ms after error IO error: lock ./data/swc_meta/meta1/LOCK: Resource temporarily unavailable
2023-02-15 01:04:15.137 [error] <0.227.0> Supervisor vmq_swc_store_sup_meta1 had child {vmq_swc_db_leveldb,meta1} started with vmq_swc_db_leveldb:start_link({swc_config,{'VerneMQ@127.0.0.1',<<239,174,60,145,53,39,215,115,159,70,176,124,80,197,199,248,97,...>>},...}, []) at undefined exit with reason {db_open,"IO error: lock ./data/swc_meta/meta1/LOCK: Resource temporarily unavailable"} in context start_error
2023-02-15 01:04:15.137 [error] <0.228.0> CRASH REPORT Process <0.228.0> with 0 neighbours exited with reason: {db_open,"IO error: lock ./data/swc_meta/meta1/LOCK: Resource temporarily unavailable"} in gen_server:init_it/6 line 835
2023-02-15 01:04:15.141 [info] <0.332.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta2" in 2000 ms after error IO error: lock ./data/swc_meta/meta2/LOCK: Resource temporarily unavailable
2023-02-15 01:04:17.143 [info] <0.332.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta2" in 2000 ms after error IO error: lock ./data/swc_meta/meta2/LOCK: Resource temporarily unavailable
2023-02-15 01:04:19.145 [info] <0.332.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta2" in 2000 ms after error IO error: lock ./data/swc_meta/meta2/LOCK: Resource temporarily unavailable
2023-02-15 01:04:21.147 [info] <0.332.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta2" in 2000 ms after error IO error: lock ./data/swc_meta/meta2/LOCK: Resource temporarily unavailable
2023-02-15 01:04:23.149 [info] <0.332.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta2" in 2000 ms after error IO error: lock ./data/swc_meta/meta2/LOCK: Resource temporarily unavailable
2023-02-15 01:04:25.150 [info] <0.332.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta2" in 2000 ms after error IO error: lock ./data/swc_meta/meta2/LOCK: Resource temporarily unavailable
2023-02-15 01:04:27.152 [info] <0.332.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta2" in 2000 ms after error IO error: lock ./data/swc_meta/meta2/LOCK: Resource temporarily unavailable
2023-02-15 01:04:29.155 [info] <0.332.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta2" in 2000 ms after error IO error: lock ./data/swc_meta/meta2/LOCK: Resource temporarily unavailable
2023-02-15 01:04:31.158 [info] <0.332.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta2" in 2000 ms after error IO error: lock ./data/swc_meta/meta2/LOCK: Resource temporarily unavailable
2023-02-15 01:04:33.161 [info] <0.332.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta2" in 2000 ms after error IO error: lock ./data/swc_meta/meta2/LOCK: Resource temporarily unavailable
2023-02-15 01:04:35.166 [info] <0.332.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta2" in 2000 ms after error IO error: lock ./data/swc_meta/meta2/LOCK: Resource temporarily unavailable
2023-02-15 01:04:37.170 [info] <0.332.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta2" in 2000 ms after error IO error: lock ./data/swc_meta/meta2/LOCK: Resource temporarily unavailable
2023-02-15 01:04:39.172 [info] <0.332.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta2" in 2000 ms after error IO error: lock ./data/swc_meta/meta2/LOCK: Resource temporarily unavailable
2023-02-15 01:04:41.175 [info] <0.332.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta2" in 2000 ms after error IO error: lock ./data/swc_meta/meta2/LOCK: Resource temporarily unavailable
2023-02-15 01:04:43.177 [info] <0.332.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta2" in 2000 ms after error IO error: lock ./data/swc_meta/meta2/LOCK: Resource temporarily unavailable
2023-02-15 01:04:45.180 [info] <0.332.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta2" in 2000 ms after error IO error: lock ./data/swc_meta/meta2/LOCK: Resource temporarily unavailable
2023-02-15 01:04:47.183 [info] <0.332.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta2" in 2000 ms after error IO error: lock ./data/swc_meta/meta2/LOCK: Resource temporarily unavailable
2023-02-15 01:04:49.186 [info] <0.332.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta2" in 2000 ms after error IO error: lock ./data/swc_meta/meta2/LOCK: Resource temporarily unavailable
2023-02-15 01:04:51.189 [info] <0.332.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta2" in 2000 ms after error IO error: lock ./data/swc_meta/meta2/LOCK: Resource temporarily unavailable
2023-02-15 01:04:53.192 [info] <0.332.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta2" in 2000 ms after error IO error: lock ./data/swc_meta/meta2/LOCK: Resource temporarily unavailable
2023-02-15 01:04:55.206 [info] <0.332.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta2" in 2000 ms after error IO error: lock ./data/swc_meta/meta2/LOCK: Resource temporarily unavailable
2023-02-15 01:04:57.209 [info] <0.332.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta2" in 2000 ms after error IO error: lock ./data/swc_meta/meta2/LOCK: Resource temporarily unavailable
2023-02-15 01:04:59.213 [info] <0.332.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta2" in 2000 ms after error IO error: lock ./data/swc_meta/meta2/LOCK: Resource temporarily unavailable
2023-02-15 01:05:01.217 [info] <0.332.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta2" in 2000 ms after error IO error: lock ./data/swc_meta/meta2/LOCK: Resource temporarily unavailable
2023-02-15 01:05:03.221 [info] <0.332.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta2" in 2000 ms after error IO error: lock ./data/swc_meta/meta2/LOCK: Resource temporarily unavailable
2023-02-15 01:05:05.224 [info] <0.332.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta2" in 2000 ms after error IO error: lock ./data/swc_meta/meta2/LOCK: Resource temporarily unavailable
2023-02-15 01:05:07.227 [info] <0.332.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta2" in 2000 ms after error IO error: lock ./data/swc_meta/meta2/LOCK: Resource temporarily unavailable
2023-02-15 01:05:09.231 [info] <0.332.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta2" in 2000 ms after error IO error: lock ./data/swc_meta/meta2/LOCK: Resource temporarily unavailable
2023-02-15 01:05:11.235 [info] <0.332.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta2" in 2000 ms after error IO error: lock ./data/swc_meta/meta2/LOCK: Resource temporarily unavailable
2023-02-15 01:05:13.239 [info] <0.332.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta2" in 2000 ms after error IO error: lock ./data/swc_meta/meta2/LOCK: Resource temporarily unavailable
2023-02-15 01:05:15.241 [error] <0.331.0> Supervisor vmq_swc_store_sup_meta2 had child {vmq_swc_db_leveldb,meta2} started with vmq_swc_db_leveldb:start_link({swc_config,{'VerneMQ@127.0.0.1',<<239,174,60,145,53,39,215,115,159,70,176,124,80,197,199,248,97,...>>},...}, []) at undefined exit with reason {db_open,"IO error: lock ./data/swc_meta/meta2/LOCK: Resource temporarily unavailable"} in context start_error
2023-02-15 01:05:15.242 [error] <0.332.0> CRASH REPORT Process <0.332.0> with 0 neighbours exited with reason: {db_open,"IO error: lock ./data/swc_meta/meta2/LOCK: Resource temporarily unavailable"} in gen_server:init_it/6 line 835
2023-02-15 01:05:15.249 [info] <0.357.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta3" in 2000 ms after error IO error: lock ./data/swc_meta/meta3/LOCK: Resource temporarily unavailable
2023-02-15 01:05:17.252 [info] <0.357.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta3" in 2000 ms after error IO error: lock ./data/swc_meta/meta3/LOCK: Resource temporarily unavailable
2023-02-15 01:05:19.253 [info] <0.357.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta3" in 2000 ms after error IO error: lock ./data/swc_meta/meta3/LOCK: Resource temporarily unavailable
2023-02-15 01:05:21.256 [info] <0.357.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta3" in 2000 ms after error IO error: lock ./data/swc_meta/meta3/LOCK: Resource temporarily unavailable
2023-02-15 01:05:23.259 [info] <0.357.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta3" in 2000 ms after error IO error: lock ./data/swc_meta/meta3/LOCK: Resource temporarily unavailable
2023-02-15 01:05:25.262 [info] <0.357.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta3" in 2000 ms after error IO error: lock ./data/swc_meta/meta3/LOCK: Resource temporarily unavailable
2023-02-15 01:05:27.264 [info] <0.357.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta3" in 2000 ms after error IO error: lock ./data/swc_meta/meta3/LOCK: Resource temporarily unavailable
2023-02-15 01:05:29.267 [info] <0.357.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta3" in 2000 ms after error IO error: lock ./data/swc_meta/meta3/LOCK: Resource temporarily unavailable
2023-02-15 01:05:31.271 [info] <0.357.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta3" in 2000 ms after error IO error: lock ./data/swc_meta/meta3/LOCK: Resource temporarily unavailable
2023-02-15 01:05:33.274 [info] <0.357.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta3" in 2000 ms after error IO error: lock ./data/swc_meta/meta3/LOCK: Resource temporarily unavailable
2023-02-15 01:05:35.278 [info] <0.357.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta3" in 2000 ms after error IO error: lock ./data/swc_meta/meta3/LOCK: Resource temporarily unavailable
2023-02-15 01:05:37.282 [info] <0.357.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta3" in 2000 ms after error IO error: lock ./data/swc_meta/meta3/LOCK: Resource temporarily unavailable
2023-02-15 01:05:39.286 [info] <0.357.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta3" in 2000 ms after error IO error: lock ./data/swc_meta/meta3/LOCK: Resource temporarily unavailable
2023-02-15 01:05:41.289 [info] <0.357.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta3" in 2000 ms after error IO error: lock ./data/swc_meta/meta3/LOCK: Resource temporarily unavailable
2023-02-15 01:05:43.292 [info] <0.357.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta3" in 2000 ms after error IO error: lock ./data/swc_meta/meta3/LOCK: Resource temporarily unavailable
2023-02-15 01:05:45.296 [info] <0.357.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta3" in 2000 ms after error IO error: lock ./data/swc_meta/meta3/LOCK: Resource temporarily unavailable
2023-02-15 01:05:47.300 [info] <0.357.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta3" in 2000 ms after error IO error: lock ./data/swc_meta/meta3/LOCK: Resource temporarily unavailable
2023-02-15 01:05:49.304 [info] <0.357.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta3" in 2000 ms after error IO error: lock ./data/swc_meta/meta3/LOCK: Resource temporarily unavailable
2023-02-15 01:05:51.307 [info] <0.357.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta3" in 2000 ms after error IO error: lock ./data/swc_meta/meta3/LOCK: Resource temporarily unavailable
2023-02-15 01:05:53.311 [info] <0.357.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta3" in 2000 ms after error IO error: lock ./data/swc_meta/meta3/LOCK: Resource temporarily unavailable
2023-02-15 01:05:55.315 [info] <0.357.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta3" in 2000 ms after error IO error: lock ./data/swc_meta/meta3/LOCK: Resource temporarily unavailable
2023-02-15 01:05:57.318 [info] <0.357.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta3" in 2000 ms after error IO error: lock ./data/swc_meta/meta3/LOCK: Resource temporarily unavailable
2023-02-15 01:05:59.322 [info] <0.357.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta3" in 2000 ms after error IO error: lock ./data/swc_meta/meta3/LOCK: Resource temporarily unavailable
2023-02-15 01:06:01.325 [info] <0.357.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta3" in 2000 ms after error IO error: lock ./data/swc_meta/meta3/LOCK: Resource temporarily unavailable
2023-02-15 01:06:03.327 [info] <0.357.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta3" in 2000 ms after error IO error: lock ./data/swc_meta/meta3/LOCK: Resource temporarily unavailable
2023-02-15 01:06:05.330 [info] <0.357.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta3" in 2000 ms after error IO error: lock ./data/swc_meta/meta3/LOCK: Resource temporarily unavailable
2023-02-15 01:06:07.335 [info] <0.357.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta3" in 2000 ms after error IO error: lock ./data/swc_meta/meta3/LOCK: Resource temporarily unavailable
2023-02-15 01:06:09.339 [info] <0.357.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta3" in 2000 ms after error IO error: lock ./data/swc_meta/meta3/LOCK: Resource temporarily unavailable
2023-02-15 01:06:11.342 [info] <0.357.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta3" in 2000 ms after error IO error: lock ./data/swc_meta/meta3/LOCK: Resource temporarily unavailable
2023-02-15 01:06:13.346 [info] <0.357.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta3" in 2000 ms after error IO error: lock ./data/swc_meta/meta3/LOCK: Resource temporarily unavailable
2023-02-15 01:06:15.348 [error] <0.356.0> Supervisor vmq_swc_store_sup_meta3 had child {vmq_swc_db_leveldb,meta3} started with vmq_swc_db_leveldb:start_link({swc_config,{'VerneMQ@127.0.0.1',<<239,174,60,145,53,39,215,115,159,70,176,124,80,197,199,248,97,...>>},...}, []) at undefined exit with reason {db_open,"IO error: lock ./data/swc_meta/meta3/LOCK: Resource temporarily unavailable"} in context start_error
2023-02-15 01:06:15.348 [error] <0.357.0> CRASH REPORT Process <0.357.0> with 0 neighbours exited with reason: {db_open,"IO error: lock ./data/swc_meta/meta3/LOCK: Resource temporarily unavailable"} in gen_server:init_it/6 line 835
2023-02-15 01:06:15.354 [info] <0.377.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta4" in 2000 ms after error IO error: lock ./data/swc_meta/meta4/LOCK: Resource temporarily unavailable
2023-02-15 01:06:17.357 [info] <0.377.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta4" in 2000 ms after error IO error: lock ./data/swc_meta/meta4/LOCK: Resource temporarily unavailable
2023-02-15 01:06:19.361 [info] <0.377.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta4" in 2000 ms after error IO error: lock ./data/swc_meta/meta4/LOCK: Resource temporarily unavailable
2023-02-15 01:06:21.364 [info] <0.377.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta4" in 2000 ms after error IO error: lock ./data/swc_meta/meta4/LOCK: Resource temporarily unavailable
2023-02-15 01:06:23.366 [info] <0.377.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta4" in 2000 ms after error IO error: lock ./data/swc_meta/meta4/LOCK: Resource temporarily unavailable
2023-02-15 01:06:25.370 [info] <0.377.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta4" in 2000 ms after error IO error: lock ./data/swc_meta/meta4/LOCK: Resource temporarily unavailable
2023-02-15 01:06:27.373 [info] <0.377.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta4" in 2000 ms after error IO error: lock ./data/swc_meta/meta4/LOCK: Resource temporarily unavailable
2023-02-15 01:06:29.375 [info] <0.377.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta4" in 2000 ms after error IO error: lock ./data/swc_meta/meta4/LOCK: Resource temporarily unavailable
2023-02-15 01:06:31.380 [info] <0.377.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta4" in 2000 ms after error IO error: lock ./data/swc_meta/meta4/LOCK: Resource temporarily unavailable
2023-02-15 01:06:33.384 [info] <0.377.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta4" in 2000 ms after error IO error: lock ./data/swc_meta/meta4/LOCK: Resource temporarily unavailable
2023-02-15 01:06:35.388 [info] <0.377.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta4" in 2000 ms after error IO error: lock ./data/swc_meta/meta4/LOCK: Resource temporarily unavailable
2023-02-15 01:06:37.393 [info] <0.377.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta4" in 2000 ms after error IO error: lock ./data/swc_meta/meta4/LOCK: Resource temporarily unavailable
2023-02-15 01:06:39.396 [info] <0.377.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta4" in 2000 ms after error IO error: lock ./data/swc_meta/meta4/LOCK: Resource temporarily unavailable
2023-02-15 01:06:41.401 [info] <0.377.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta4" in 2000 ms after error IO error: lock ./data/swc_meta/meta4/LOCK: Resource temporarily unavailable
2023-02-15 01:06:43.403 [info] <0.377.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta4" in 2000 ms after error IO error: lock ./data/swc_meta/meta4/LOCK: Resource temporarily unavailable
2023-02-15 01:06:45.407 [info] <0.377.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta4" in 2000 ms after error IO error: lock ./data/swc_meta/meta4/LOCK: Resource temporarily unavailable
2023-02-15 01:06:47.410 [info] <0.377.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta4" in 2000 ms after error IO error: lock ./data/swc_meta/meta4/LOCK: Resource temporarily unavailable
2023-02-15 01:06:49.414 [info] <0.377.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta4" in 2000 ms after error IO error: lock ./data/swc_meta/meta4/LOCK: Resource temporarily unavailable
2023-02-15 01:06:51.417 [info] <0.377.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta4" in 2000 ms after error IO error: lock ./data/swc_meta/meta4/LOCK: Resource temporarily unavailable
2023-02-15 01:06:53.421 [info] <0.377.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta4" in 2000 ms after error IO error: lock ./data/swc_meta/meta4/LOCK: Resource temporarily unavailable
2023-02-15 01:06:55.426 [info] <0.377.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta4" in 2000 ms after error IO error: lock ./data/swc_meta/meta4/LOCK: Resource temporarily unavailable
2023-02-15 01:06:57.429 [info] <0.377.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta4" in 2000 ms after error IO error: lock ./data/swc_meta/meta4/LOCK: Resource temporarily unavailable
2023-02-15 01:06:59.431 [info] <0.377.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta4" in 2000 ms after error IO error: lock ./data/swc_meta/meta4/LOCK: Resource temporarily unavailable
2023-02-15 01:07:01.432 [info] <0.377.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta4" in 2000 ms after error IO error: lock ./data/swc_meta/meta4/LOCK: Resource temporarily unavailable
2023-02-15 01:07:03.436 [info] <0.377.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta4" in 2000 ms after error IO error: lock ./data/swc_meta/meta4/LOCK: Resource temporarily unavailable
2023-02-15 01:07:05.438 [info] <0.377.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta4" in 2000 ms after error IO error: lock ./data/swc_meta/meta4/LOCK: Resource temporarily unavailable
2023-02-15 01:07:07.441 [info] <0.377.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta4" in 2000 ms after error IO error: lock ./data/swc_meta/meta4/LOCK: Resource temporarily unavailable
2023-02-15 01:07:09.444 [info] <0.377.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta4" in 2000 ms after error IO error: lock ./data/swc_meta/meta4/LOCK: Resource temporarily unavailable
2023-02-15 01:07:11.447 [info] <0.377.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta4" in 2000 ms after error IO error: lock ./data/swc_meta/meta4/LOCK: Resource temporarily unavailable
2023-02-15 01:07:13.451 [info] <0.377.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta4" in 2000 ms after error IO error: lock ./data/swc_meta/meta4/LOCK: Resource temporarily unavailable
2023-02-15 01:07:15.453 [error] <0.376.0> Supervisor vmq_swc_store_sup_meta4 had child {vmq_swc_db_leveldb,meta4} started with vmq_swc_db_leveldb:start_link({swc_config,{'VerneMQ@127.0.0.1',<<239,174,60,145,53,39,215,115,159,70,176,124,80,197,199,248,97,...>>},...}, []) at undefined exit with reason {db_open,"IO error: lock ./data/swc_meta/meta4/LOCK: Resource temporarily unavailable"} in context start_error
2023-02-15 01:07:15.453 [error] <0.377.0> CRASH REPORT Process <0.377.0> with 0 neighbours exited with reason: {db_open,"IO error: lock ./data/swc_meta/meta4/LOCK: Resource temporarily unavailable"} in gen_server:init_it/6 line 835
2023-02-15 01:07:15.458 [info] <0.388.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta5" in 2000 ms after error IO error: lock ./data/swc_meta/meta5/LOCK: Resource temporarily unavailable
2023-02-15 01:07:17.462 [info] <0.388.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta5" in 2000 ms after error IO error: lock ./data/swc_meta/meta5/LOCK: Resource temporarily unavailable
2023-02-15 01:07:19.465 [info] <0.388.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta5" in 2000 ms after error IO error: lock ./data/swc_meta/meta5/LOCK: Resource temporarily unavailable
2023-02-15 01:07:21.469 [info] <0.388.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta5" in 2000 ms after error IO error: lock ./data/swc_meta/meta5/LOCK: Resource temporarily unavailable
2023-02-15 01:07:23.473 [info] <0.388.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta5" in 2000 ms after error IO error: lock ./data/swc_meta/meta5/LOCK: Resource temporarily unavailable
2023-02-15 01:07:25.476 [info] <0.388.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta5" in 2000 ms after error IO error: lock ./data/swc_meta/meta5/LOCK: Resource temporarily unavailable
2023-02-15 01:07:27.480 [info] <0.388.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta5" in 2000 ms after error IO error: lock ./data/swc_meta/meta5/LOCK: Resource temporarily unavailable
2023-02-15 01:07:29.483 [info] <0.388.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta5" in 2000 ms after error IO error: lock ./data/swc_meta/meta5/LOCK: Resource temporarily unavailable
2023-02-15 01:07:31.486 [info] <0.388.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta5" in 2000 ms after error IO error: lock ./data/swc_meta/meta5/LOCK: Resource temporarily unavailable
2023-02-15 01:07:33.491 [info] <0.388.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta5" in 2000 ms after error IO error: lock ./data/swc_meta/meta5/LOCK: Resource temporarily unavailable
2023-02-15 01:07:35.494 [info] <0.388.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta5" in 2000 ms after error IO error: lock ./data/swc_meta/meta5/LOCK: Resource temporarily unavailable
2023-02-15 01:07:37.497 [info] <0.388.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta5" in 2000 ms after error IO error: lock ./data/swc_meta/meta5/LOCK: Resource temporarily unavailable
2023-02-15 01:07:39.500 [info] <0.388.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta5" in 2000 ms after error IO error: lock ./data/swc_meta/meta5/LOCK: Resource temporarily unavailable
2023-02-15 01:07:41.501 [info] <0.388.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta5" in 2000 ms after error IO error: lock ./data/swc_meta/meta5/LOCK: Resource temporarily unavailable
2023-02-15 01:07:43.504 [info] <0.388.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta5" in 2000 ms after error IO error: lock ./data/swc_meta/meta5/LOCK: Resource temporarily unavailable
2023-02-15 01:07:45.506 [info] <0.388.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta5" in 2000 ms after error IO error: lock ./data/swc_meta/meta5/LOCK: Resource temporarily unavailable
2023-02-15 01:07:47.510 [info] <0.388.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta5" in 2000 ms after error IO error: lock ./data/swc_meta/meta5/LOCK: Resource temporarily unavailable
2023-02-15 01:07:49.513 [info] <0.388.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta5" in 2000 ms after error IO error: lock ./data/swc_meta/meta5/LOCK: Resource temporarily unavailable
2023-02-15 01:07:51.516 [info] <0.388.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta5" in 2000 ms after error IO error: lock ./data/swc_meta/meta5/LOCK: Resource temporarily unavailable
2023-02-15 01:07:53.519 [info] <0.388.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta5" in 2000 ms after error IO error: lock ./data/swc_meta/meta5/LOCK: Resource temporarily unavailable
2023-02-15 01:07:55.521 [info] <0.388.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta5" in 2000 ms after error IO error: lock ./data/swc_meta/meta5/LOCK: Resource temporarily unavailable
2023-02-15 01:07:57.524 [info] <0.388.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta5" in 2000 ms after error IO error: lock ./data/swc_meta/meta5/LOCK: Resource temporarily unavailable
2023-02-15 01:07:59.528 [info] <0.388.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta5" in 2000 ms after error IO error: lock ./data/swc_meta/meta5/LOCK: Resource temporarily unavailable
2023-02-15 01:08:01.532 [info] <0.388.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta5" in 2000 ms after error IO error: lock ./data/swc_meta/meta5/LOCK: Resource temporarily unavailable
2023-02-15 01:08:03.536 [info] <0.388.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta5" in 2000 ms after error IO error: lock ./data/swc_meta/meta5/LOCK: Resource temporarily unavailable
2023-02-15 01:08:05.540 [info] <0.388.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta5" in 2000 ms after error IO error: lock ./data/swc_meta/meta5/LOCK: Resource temporarily unavailable
2023-02-15 01:08:07.545 [info] <0.388.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta5" in 2000 ms after error IO error: lock ./data/swc_meta/meta5/LOCK: Resource temporarily unavailable
2023-02-15 01:08:09.548 [info] <0.388.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta5" in 2000 ms after error IO error: lock ./data/swc_meta/meta5/LOCK: Resource temporarily unavailable
2023-02-15 01:08:11.551 [info] <0.388.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta5" in 2000 ms after error IO error: lock ./data/swc_meta/meta5/LOCK: Resource temporarily unavailable
2023-02-15 01:08:13.553 [info] <0.388.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta5" in 2000 ms after error IO error: lock ./data/swc_meta/meta5/LOCK: Resource temporarily unavailable
2023-02-15 01:08:15.554 [error] <0.387.0> Supervisor vmq_swc_store_sup_meta5 had child {vmq_swc_db_leveldb,meta5} started with vmq_swc_db_leveldb:start_link({swc_config,{'VerneMQ@127.0.0.1',<<239,174,60,145,53,39,215,115,159,70,176,124,80,197,199,248,97,...>>},...}, []) at undefined exit with reason {db_open,"IO error: lock ./data/swc_meta/meta5/LOCK: Resource temporarily unavailable"} in context start_error
2023-02-15 01:08:15.554 [error] <0.388.0> CRASH REPORT Process <0.388.0> with 0 neighbours exited with reason: {db_open,"IO error: lock ./data/swc_meta/meta5/LOCK: Resource temporarily unavailable"} in gen_server:init_it/6 line 835
2023-02-15 01:08:15.561 [info] <0.398.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta6" in 2000 ms after error IO error: lock ./data/swc_meta/meta6/LOCK: Resource temporarily unavailable
2023-02-15 01:08:17.565 [info] <0.398.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta6" in 2000 ms after error IO error: lock ./data/swc_meta/meta6/LOCK: Resource temporarily unavailable
2023-02-15 01:08:19.569 [info] <0.398.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta6" in 2000 ms after error IO error: lock ./data/swc_meta/meta6/LOCK: Resource temporarily unavailable
2023-02-15 01:08:21.572 [info] <0.398.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta6" in 2000 ms after error IO error: lock ./data/swc_meta/meta6/LOCK: Resource temporarily unavailable
2023-02-15 01:08:23.576 [info] <0.398.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta6" in 2000 ms after error IO error: lock ./data/swc_meta/meta6/LOCK: Resource temporarily unavailable
2023-02-15 01:08:25.579 [info] <0.398.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta6" in 2000 ms after error IO error: lock ./data/swc_meta/meta6/LOCK: Resource temporarily unavailable
2023-02-15 01:08:27.580 [info] <0.398.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta6" in 2000 ms after error IO error: lock ./data/swc_meta/meta6/LOCK: Resource temporarily unavailable
2023-02-15 01:08:29.584 [info] <0.398.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta6" in 2000 ms after error IO error: lock ./data/swc_meta/meta6/LOCK: Resource temporarily unavailable
2023-02-15 01:08:31.585 [info] <0.398.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta6" in 2000 ms after error IO error: lock ./data/swc_meta/meta6/LOCK: Resource temporarily unavailable
2023-02-15 01:08:33.588 [info] <0.398.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta6" in 2000 ms after error IO error: lock ./data/swc_meta/meta6/LOCK: Resource temporarily unavailable
2023-02-15 01:08:35.591 [info] <0.398.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta6" in 2000 ms after error IO error: lock ./data/swc_meta/meta6/LOCK: Resource temporarily unavailable
2023-02-15 01:08:37.595 [info] <0.398.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta6" in 2000 ms after error IO error: lock ./data/swc_meta/meta6/LOCK: Resource temporarily unavailable
2023-02-15 01:08:39.598 [info] <0.398.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta6" in 2000 ms after error IO error: lock ./data/swc_meta/meta6/LOCK: Resource temporarily unavailable
2023-02-15 01:08:41.602 [info] <0.398.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta6" in 2000 ms after error IO error: lock ./data/swc_meta/meta6/LOCK: Resource temporarily unavailable
2023-02-15 01:08:43.603 [info] <0.398.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta6" in 2000 ms after error IO error: lock ./data/swc_meta/meta6/LOCK: Resource temporarily unavailable
2023-02-15 01:08:45.606 [info] <0.398.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta6" in 2000 ms after error IO error: lock ./data/swc_meta/meta6/LOCK: Resource temporarily unavailable
2023-02-15 01:08:47.608 [info] <0.398.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta6" in 2000 ms after error IO error: lock ./data/swc_meta/meta6/LOCK: Resource temporarily unavailable
2023-02-15 01:08:49.610 [info] <0.398.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta6" in 2000 ms after error IO error: lock ./data/swc_meta/meta6/LOCK: Resource temporarily unavailable
2023-02-15 01:08:51.612 [info] <0.398.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta6" in 2000 ms after error IO error: lock ./data/swc_meta/meta6/LOCK: Resource temporarily unavailable
2023-02-15 01:08:53.616 [info] <0.398.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta6" in 2000 ms after error IO error: lock ./data/swc_meta/meta6/LOCK: Resource temporarily unavailable
2023-02-15 01:08:55.619 [info] <0.398.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta6" in 2000 ms after error IO error: lock ./data/swc_meta/meta6/LOCK: Resource temporarily unavailable
2023-02-15 01:08:57.622 [info] <0.398.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta6" in 2000 ms after error IO error: lock ./data/swc_meta/meta6/LOCK: Resource temporarily unavailable
2023-02-15 01:08:59.624 [info] <0.398.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta6" in 2000 ms after error IO error: lock ./data/swc_meta/meta6/LOCK: Resource temporarily unavailable
2023-02-15 01:09:01.627 [info] <0.398.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta6" in 2000 ms after error IO error: lock ./data/swc_meta/meta6/LOCK: Resource temporarily unavailable
2023-02-15 01:09:03.629 [info] <0.398.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta6" in 2000 ms after error IO error: lock ./data/swc_meta/meta6/LOCK: Resource temporarily unavailable
2023-02-15 01:09:05.632 [info] <0.398.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta6" in 2000 ms after error IO error: lock ./data/swc_meta/meta6/LOCK: Resource temporarily unavailable
2023-02-15 01:09:07.635 [info] <0.398.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta6" in 2000 ms after error IO error: lock ./data/swc_meta/meta6/LOCK: Resource temporarily unavailable
2023-02-15 01:09:09.638 [info] <0.398.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta6" in 2000 ms after error IO error: lock ./data/swc_meta/meta6/LOCK: Resource temporarily unavailable
2023-02-15 01:09:11.641 [info] <0.398.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta6" in 2000 ms after error IO error: lock ./data/swc_meta/meta6/LOCK: Resource temporarily unavailable
2023-02-15 01:09:13.646 [info] <0.398.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta6" in 2000 ms after error IO error: lock ./data/swc_meta/meta6/LOCK: Resource temporarily unavailable
2023-02-15 01:09:15.648 [error] <0.397.0> Supervisor vmq_swc_store_sup_meta6 had child {vmq_swc_db_leveldb,meta6} started with vmq_swc_db_leveldb:start_link({swc_config,{'VerneMQ@127.0.0.1',<<239,174,60,145,53,39,215,115,159,70,176,124,80,197,199,248,97,...>>},...}, []) at undefined exit with reason {db_open,"IO error: lock ./data/swc_meta/meta6/LOCK: Resource temporarily unavailable"} in context start_error
2023-02-15 01:09:15.648 [error] <0.398.0> CRASH REPORT Process <0.398.0> with 0 neighbours exited with reason: {db_open,"IO error: lock ./data/swc_meta/meta6/LOCK: Resource temporarily unavailable"} in gen_server:init_it/6 line 835
2023-02-15 01:09:15.654 [info] <0.412.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta7" in 2000 ms after error IO error: lock ./data/swc_meta/meta7/LOCK: Resource temporarily unavailable
2023-02-15 01:09:17.657 [info] <0.412.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta7" in 2000 ms after error IO error: lock ./data/swc_meta/meta7/LOCK: Resource temporarily unavailable
2023-02-15 01:09:19.661 [info] <0.412.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta7" in 2000 ms after error IO error: lock ./data/swc_meta/meta7/LOCK: Resource temporarily unavailable
2023-02-15 01:09:21.665 [info] <0.412.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta7" in 2000 ms after error IO error: lock ./data/swc_meta/meta7/LOCK: Resource temporarily unavailable
2023-02-15 01:09:23.668 [info] <0.412.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta7" in 2000 ms after error IO error: lock ./data/swc_meta/meta7/LOCK: Resource temporarily unavailable
2023-02-15 01:09:25.669 [info] <0.412.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta7" in 2000 ms after error IO error: lock ./data/swc_meta/meta7/LOCK: Resource temporarily unavailable
2023-02-15 01:09:27.673 [info] <0.412.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta7" in 2000 ms after error IO error: lock ./data/swc_meta/meta7/LOCK: Resource temporarily unavailable
2023-02-15 01:09:29.677 [info] <0.412.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta7" in 2000 ms after error IO error: lock ./data/swc_meta/meta7/LOCK: Resource temporarily unavailable
2023-02-15 01:09:31.680 [info] <0.412.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta7" in 2000 ms after error IO error: lock ./data/swc_meta/meta7/LOCK: Resource temporarily unavailable
2023-02-15 01:09:33.684 [info] <0.412.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta7" in 2000 ms after error IO error: lock ./data/swc_meta/meta7/LOCK: Resource temporarily unavailable
2023-02-15 01:09:35.687 [info] <0.412.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta7" in 2000 ms after error IO error: lock ./data/swc_meta/meta7/LOCK: Resource temporarily unavailable
2023-02-15 01:09:37.689 [info] <0.412.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta7" in 2000 ms after error IO error: lock ./data/swc_meta/meta7/LOCK: Resource temporarily unavailable
2023-02-15 01:09:39.692 [info] <0.412.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta7" in 2000 ms after error IO error: lock ./data/swc_meta/meta7/LOCK: Resource temporarily unavailable
2023-02-15 01:09:41.695 [info] <0.412.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta7" in 2000 ms after error IO error: lock ./data/swc_meta/meta7/LOCK: Resource temporarily unavailable
2023-02-15 01:09:43.698 [info] <0.412.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta7" in 2000 ms after error IO error: lock ./data/swc_meta/meta7/LOCK: Resource temporarily unavailable
2023-02-15 01:09:45.702 [info] <0.412.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta7" in 2000 ms after error IO error: lock ./data/swc_meta/meta7/LOCK: Resource temporarily unavailable
2023-02-15 01:09:47.705 [info] <0.412.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta7" in 2000 ms after error IO error: lock ./data/swc_meta/meta7/LOCK: Resource temporarily unavailable
2023-02-15 01:09:49.711 [info] <0.412.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta7" in 2000 ms after error IO error: lock ./data/swc_meta/meta7/LOCK: Resource temporarily unavailable
2023-02-15 01:09:51.714 [info] <0.412.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta7" in 2000 ms after error IO error: lock ./data/swc_meta/meta7/LOCK: Resource temporarily unavailable
2023-02-15 01:09:53.718 [info] <0.412.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta7" in 2000 ms after error IO error: lock ./data/swc_meta/meta7/LOCK: Resource temporarily unavailable
2023-02-15 01:09:55.722 [info] <0.412.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta7" in 2000 ms after error IO error: lock ./data/swc_meta/meta7/LOCK: Resource temporarily unavailable
2023-02-15 01:09:57.726 [info] <0.412.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta7" in 2000 ms after error IO error: lock ./data/swc_meta/meta7/LOCK: Resource temporarily unavailable
2023-02-15 01:09:59.730 [info] <0.412.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta7" in 2000 ms after error IO error: lock ./data/swc_meta/meta7/LOCK: Resource temporarily unavailable
2023-02-15 01:10:01.733 [info] <0.412.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta7" in 2000 ms after error IO error: lock ./data/swc_meta/meta7/LOCK: Resource temporarily unavailable
2023-02-15 01:10:03.737 [info] <0.412.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta7" in 2000 ms after error IO error: lock ./data/swc_meta/meta7/LOCK: Resource temporarily unavailable
2023-02-15 01:10:05.741 [info] <0.412.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta7" in 2000 ms after error IO error: lock ./data/swc_meta/meta7/LOCK: Resource temporarily unavailable
2023-02-15 01:10:07.746 [info] <0.412.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta7" in 2000 ms after error IO error: lock ./data/swc_meta/meta7/LOCK: Resource temporarily unavailable
2023-02-15 01:10:09.749 [info] <0.412.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta7" in 2000 ms after error IO error: lock ./data/swc_meta/meta7/LOCK: Resource temporarily unavailable
2023-02-15 01:10:11.752 [info] <0.412.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta7" in 2000 ms after error IO error: lock ./data/swc_meta/meta7/LOCK: Resource temporarily unavailable
2023-02-15 01:10:13.758 [info] <0.412.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta7" in 2000 ms after error IO error: lock ./data/swc_meta/meta7/LOCK: Resource temporarily unavailable
2023-02-15 01:10:15.760 [error] <0.411.0> Supervisor vmq_swc_store_sup_meta7 had child {vmq_swc_db_leveldb,meta7} started with vmq_swc_db_leveldb:start_link({swc_config,{'VerneMQ@127.0.0.1',<<239,174,60,145,53,39,215,115,159,70,176,124,80,197,199,248,97,...>>},...}, []) at undefined exit with reason {db_open,"IO error: lock ./data/swc_meta/meta7/LOCK: Resource temporarily unavailable"} in context start_error
2023-02-15 01:10:15.760 [error] <0.412.0> CRASH REPORT Process <0.412.0> with 0 neighbours exited with reason: {db_open,"IO error: lock ./data/swc_meta/meta7/LOCK: Resource temporarily unavailable"} in gen_server:init_it/6 line 835
2023-02-15 01:10:15.769 [info] <0.422.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta8" in 2000 ms after error IO error: lock ./data/swc_meta/meta8/LOCK: Resource temporarily unavailable
2023-02-15 01:10:17.772 [info] <0.422.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta8" in 2000 ms after error IO error: lock ./data/swc_meta/meta8/LOCK: Resource temporarily unavailable
2023-02-15 01:10:19.775 [info] <0.422.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta8" in 2000 ms after error IO error: lock ./data/swc_meta/meta8/LOCK: Resource temporarily unavailable
2023-02-15 01:10:21.778 [info] <0.422.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta8" in 2000 ms after error IO error: lock ./data/swc_meta/meta8/LOCK: Resource temporarily unavailable
2023-02-15 01:10:23.783 [info] <0.422.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta8" in 2000 ms after error IO error: lock ./data/swc_meta/meta8/LOCK: Resource temporarily unavailable
2023-02-15 01:10:25.787 [info] <0.422.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta8" in 2000 ms after error IO error: lock ./data/swc_meta/meta8/LOCK: Resource temporarily unavailable
2023-02-15 01:10:27.791 [info] <0.422.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta8" in 2000 ms after error IO error: lock ./data/swc_meta/meta8/LOCK: Resource temporarily unavailable
2023-02-15 01:10:29.794 [info] <0.422.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta8" in 2000 ms after error IO error: lock ./data/swc_meta/meta8/LOCK: Resource temporarily unavailable
2023-02-15 01:10:31.799 [info] <0.422.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta8" in 2000 ms after error IO error: lock ./data/swc_meta/meta8/LOCK: Resource temporarily unavailable
2023-02-15 01:10:33.802 [info] <0.422.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta8" in 2000 ms after error IO error: lock ./data/swc_meta/meta8/LOCK: Resource temporarily unavailable
2023-02-15 01:10:35.806 [info] <0.422.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta8" in 2000 ms after error IO error: lock ./data/swc_meta/meta8/LOCK: Resource temporarily unavailable
2023-02-15 01:10:37.810 [info] <0.422.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta8" in 2000 ms after error IO error: lock ./data/swc_meta/meta8/LOCK: Resource temporarily unavailable
2023-02-15 01:10:39.815 [info] <0.422.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta8" in 2000 ms after error IO error: lock ./data/swc_meta/meta8/LOCK: Resource temporarily unavailable
2023-02-15 01:10:41.818 [info] <0.422.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta8" in 2000 ms after error IO error: lock ./data/swc_meta/meta8/LOCK: Resource temporarily unavailable
2023-02-15 01:10:43.822 [info] <0.422.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta8" in 2000 ms after error IO error: lock ./data/swc_meta/meta8/LOCK: Resource temporarily unavailable
2023-02-15 01:10:45.820 [info] <0.422.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta8" in 2000 ms after error IO error: lock ./data/swc_meta/meta8/LOCK: Resource temporarily unavailable
2023-02-15 01:10:47.821 [info] <0.422.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta8" in 2000 ms after error IO error: lock ./data/swc_meta/meta8/LOCK: Resource temporarily unavailable
2023-02-15 01:10:49.821 [info] <0.422.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta8" in 2000 ms after error IO error: lock ./data/swc_meta/meta8/LOCK: Resource temporarily unavailable
2023-02-15 01:10:51.819 [info] <0.422.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta8" in 2000 ms after error IO error: lock ./data/swc_meta/meta8/LOCK: Resource temporarily unavailable
2023-02-15 01:10:53.820 [info] <0.422.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta8" in 2000 ms after error IO error: lock ./data/swc_meta/meta8/LOCK: Resource temporarily unavailable
2023-02-15 01:10:55.820 [info] <0.422.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta8" in 2000 ms after error IO error: lock ./data/swc_meta/meta8/LOCK: Resource temporarily unavailable
2023-02-15 01:10:57.823 [info] <0.422.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta8" in 2000 ms after error IO error: lock ./data/swc_meta/meta8/LOCK: Resource temporarily unavailable
2023-02-15 01:10:59.824 [info] <0.422.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta8" in 2000 ms after error IO error: lock ./data/swc_meta/meta8/LOCK: Resource temporarily unavailable
2023-02-15 01:11:01.824 [info] <0.422.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta8" in 2000 ms after error IO error: lock ./data/swc_meta/meta8/LOCK: Resource temporarily unavailable
2023-02-15 01:11:03.826 [info] <0.422.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta8" in 2000 ms after error IO error: lock ./data/swc_meta/meta8/LOCK: Resource temporarily unavailable
2023-02-15 01:11:05.829 [info] <0.422.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta8" in 2000 ms after error IO error: lock ./data/swc_meta/meta8/LOCK: Resource temporarily unavailable
2023-02-15 01:11:07.832 [info] <0.422.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta8" in 2000 ms after error IO error: lock ./data/swc_meta/meta8/LOCK: Resource temporarily unavailable
2023-02-15 01:11:09.835 [info] <0.422.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta8" in 2000 ms after error IO error: lock ./data/swc_meta/meta8/LOCK: Resource temporarily unavailable
2023-02-15 01:11:11.836 [info] <0.422.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta8" in 2000 ms after error IO error: lock ./data/swc_meta/meta8/LOCK: Resource temporarily unavailable
2023-02-15 01:11:13.839 [info] <0.422.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta8" in 2000 ms after error IO error: lock ./data/swc_meta/meta8/LOCK: Resource temporarily unavailable
2023-02-15 01:11:15.841 [error] <0.421.0> Supervisor vmq_swc_store_sup_meta8 had child {vmq_swc_db_leveldb,meta8} started with vmq_swc_db_leveldb:start_link({swc_config,{'VerneMQ@127.0.0.1',<<239,174,60,145,53,39,215,115,159,70,176,124,80,197,199,248,97,...>>},...}, []) at undefined exit with reason {db_open,"IO error: lock ./data/swc_meta/meta8/LOCK: Resource temporarily unavailable"} in context start_error
2023-02-15 01:11:15.841 [error] <0.422.0> CRASH REPORT Process <0.422.0> with 0 neighbours exited with reason: {db_open,"IO error: lock ./data/swc_meta/meta8/LOCK: Resource temporarily unavailable"} in gen_server:init_it/6 line 835
2023-02-15 01:11:15.846 [info] <0.433.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta9" in 2000 ms after error IO error: lock ./data/swc_meta/meta9/LOCK: Resource temporarily unavailable
2023-02-15 01:11:17.866 [info] <0.433.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta9" in 2000 ms after error IO error: lock ./data/swc_meta/meta9/LOCK: Resource temporarily unavailable
2023-02-15 01:11:19.888 [info] <0.433.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta9" in 2000 ms after error IO error: lock ./data/swc_meta/meta9/LOCK: Resource temporarily unavailable
2023-02-15 01:11:21.894 [info] <0.433.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta9" in 2000 ms after error IO error: lock ./data/swc_meta/meta9/LOCK: Resource temporarily unavailable
2023-02-15 01:11:23.897 [info] <0.433.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta9" in 2000 ms after error IO error: lock ./data/swc_meta/meta9/LOCK: Resource temporarily unavailable
2023-02-15 01:11:25.900 [info] <0.433.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta9" in 2000 ms after error IO error: lock ./data/swc_meta/meta9/LOCK: Resource temporarily unavailable
2023-02-15 01:11:27.903 [info] <0.433.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta9" in 2000 ms after error IO error: lock ./data/swc_meta/meta9/LOCK: Resource temporarily unavailable
2023-02-15 01:11:29.907 [info] <0.433.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta9" in 2000 ms after error IO error: lock ./data/swc_meta/meta9/LOCK: Resource temporarily unavailable
2023-02-15 01:11:31.908 [info] <0.433.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta9" in 2000 ms after error IO error: lock ./data/swc_meta/meta9/LOCK: Resource temporarily unavailable
2023-02-15 01:11:33.912 [info] <0.433.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta9" in 2000 ms after error IO error: lock ./data/swc_meta/meta9/LOCK: Resource temporarily unavailable
2023-02-15 01:11:35.918 [info] <0.433.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta9" in 2000 ms after error IO error: lock ./data/swc_meta/meta9/LOCK: Resource temporarily unavailable
2023-02-15 01:11:37.921 [info] <0.433.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta9" in 2000 ms after error IO error: lock ./data/swc_meta/meta9/LOCK: Resource temporarily unavailable
2023-02-15 01:11:39.925 [info] <0.433.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta9" in 2000 ms after error IO error: lock ./data/swc_meta/meta9/LOCK: Resource temporarily unavailable
2023-02-15 01:11:41.929 [info] <0.433.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta9" in 2000 ms after error IO error: lock ./data/swc_meta/meta9/LOCK: Resource temporarily unavailable
2023-02-15 01:11:43.933 [info] <0.433.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta9" in 2000 ms after error IO error: lock ./data/swc_meta/meta9/LOCK: Resource temporarily unavailable
2023-02-15 01:11:45.936 [info] <0.433.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta9" in 2000 ms after error IO error: lock ./data/swc_meta/meta9/LOCK: Resource temporarily unavailable
2023-02-15 01:11:47.939 [info] <0.433.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta9" in 2000 ms after error IO error: lock ./data/swc_meta/meta9/LOCK: Resource temporarily unavailable
2023-02-15 01:11:49.942 [info] <0.433.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta9" in 2000 ms after error IO error: lock ./data/swc_meta/meta9/LOCK: Resource temporarily unavailable
2023-02-15 01:11:51.947 [info] <0.433.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta9" in 2000 ms after error IO error: lock ./data/swc_meta/meta9/LOCK: Resource temporarily unavailable
2023-02-15 01:11:53.951 [info] <0.433.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta9" in 2000 ms after error IO error: lock ./data/swc_meta/meta9/LOCK: Resource temporarily unavailable
2023-02-15 01:11:55.952 [info] <0.433.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta9" in 2000 ms after error IO error: lock ./data/swc_meta/meta9/LOCK: Resource temporarily unavailable
2023-02-15 01:11:57.956 [info] <0.433.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta9" in 2000 ms after error IO error: lock ./data/swc_meta/meta9/LOCK: Resource temporarily unavailable
2023-02-15 01:11:59.959 [info] <0.433.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta9" in 2000 ms after error IO error: lock ./data/swc_meta/meta9/LOCK: Resource temporarily unavailable
2023-02-15 01:12:01.964 [info] <0.433.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta9" in 2000 ms after error IO error: lock ./data/swc_meta/meta9/LOCK: Resource temporarily unavailable
2023-02-15 01:12:03.968 [info] <0.433.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta9" in 2000 ms after error IO error: lock ./data/swc_meta/meta9/LOCK: Resource temporarily unavailable
2023-02-15 01:12:05.972 [info] <0.433.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta9" in 2000 ms after error IO error: lock ./data/swc_meta/meta9/LOCK: Resource temporarily unavailable
2023-02-15 01:12:07.976 [info] <0.433.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta9" in 2000 ms after error IO error: lock ./data/swc_meta/meta9/LOCK: Resource temporarily unavailable
2023-02-15 01:12:09.980 [info] <0.433.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta9" in 2000 ms after error IO error: lock ./data/swc_meta/meta9/LOCK: Resource temporarily unavailable
2023-02-15 01:12:11.983 [info] <0.433.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta9" in 2000 ms after error IO error: lock ./data/swc_meta/meta9/LOCK: Resource temporarily unavailable
2023-02-15 01:12:13.985 [info] <0.433.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta9" in 2000 ms after error IO error: lock ./data/swc_meta/meta9/LOCK: Resource temporarily unavailable
2023-02-15 01:12:15.987 [error] <0.432.0> Supervisor vmq_swc_store_sup_meta9 had child {vmq_swc_db_leveldb,meta9} started with vmq_swc_db_leveldb:start_link({swc_config,{'VerneMQ@127.0.0.1',<<239,174,60,145,53,39,215,115,159,70,176,124,80,197,199,248,97,...>>},...}, []) at undefined exit with reason {db_open,"IO error: lock ./data/swc_meta/meta9/LOCK: Resource temporarily unavailable"} in context start_error
2023-02-15 01:12:15.988 [error] <0.433.0> CRASH REPORT Process <0.433.0> with 0 neighbours exited with reason: {db_open,"IO error: lock ./data/swc_meta/meta9/LOCK: Resource temporarily unavailable"} in gen_server:init_it/6 line 835
2023-02-15 01:12:15.996 [info] <0.443.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta10" in 2000 ms after error IO error: lock ./data/swc_meta/meta10/LOCK: Resource temporarily unavailable
2023-02-15 01:12:17.997 [info] <0.443.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta10" in 2000 ms after error IO error: lock ./data/swc_meta/meta10/LOCK: Resource temporarily unavailable
2023-02-15 01:12:20.001 [info] <0.443.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta10" in 2000 ms after error IO error: lock ./data/swc_meta/meta10/LOCK: Resource temporarily unavailable
2023-02-15 01:12:22.005 [info] <0.443.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta10" in 2000 ms after error IO error: lock ./data/swc_meta/meta10/LOCK: Resource temporarily unavailable
2023-02-15 01:12:24.008 [info] <0.443.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta10" in 2000 ms after error IO error: lock ./data/swc_meta/meta10/LOCK: Resource temporarily unavailable
2023-02-15 01:12:26.012 [info] <0.443.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta10" in 2000 ms after error IO error: lock ./data/swc_meta/meta10/LOCK: Resource temporarily unavailable
2023-02-15 01:12:28.013 [info] <0.443.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta10" in 2000 ms after error IO error: lock ./data/swc_meta/meta10/LOCK: Resource temporarily unavailable
2023-02-15 01:12:30.017 [info] <0.443.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta10" in 2000 ms after error IO error: lock ./data/swc_meta/meta10/LOCK: Resource temporarily unavailable
2023-02-15 01:12:32.019 [info] <0.443.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta10" in 2000 ms after error IO error: lock ./data/swc_meta/meta10/LOCK: Resource temporarily unavailable
2023-02-15 01:12:34.023 [info] <0.443.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta10" in 2000 ms after error IO error: lock ./data/swc_meta/meta10/LOCK: Resource temporarily unavailable
2023-02-15 01:12:36.027 [info] <0.443.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta10" in 2000 ms after error IO error: lock ./data/swc_meta/meta10/LOCK: Resource temporarily unavailable
2023-02-15 01:12:38.030 [info] <0.443.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta10" in 2000 ms after error IO error: lock ./data/swc_meta/meta10/LOCK: Resource temporarily unavailable
2023-02-15 01:12:40.034 [info] <0.443.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta10" in 2000 ms after error IO error: lock ./data/swc_meta/meta10/LOCK: Resource temporarily unavailable
2023-02-15 01:12:42.037 [info] <0.443.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta10" in 2000 ms after error IO error: lock ./data/swc_meta/meta10/LOCK: Resource temporarily unavailable
2023-02-15 01:12:44.040 [info] <0.443.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta10" in 2000 ms after error IO error: lock ./data/swc_meta/meta10/LOCK: Resource temporarily unavailable
2023-02-15 01:12:46.044 [info] <0.443.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta10" in 2000 ms after error IO error: lock ./data/swc_meta/meta10/LOCK: Resource temporarily unavailable
2023-02-15 01:12:48.048 [info] <0.443.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta10" in 2000 ms after error IO error: lock ./data/swc_meta/meta10/LOCK: Resource temporarily unavailable
2023-02-15 01:12:50.052 [info] <0.443.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta10" in 2000 ms after error IO error: lock ./data/swc_meta/meta10/LOCK: Resource temporarily unavailable
2023-02-15 01:12:52.056 [info] <0.443.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta10" in 2000 ms after error IO error: lock ./data/swc_meta/meta10/LOCK: Resource temporarily unavailable
2023-02-15 01:12:54.060 [info] <0.443.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta10" in 2000 ms after error IO error: lock ./data/swc_meta/meta10/LOCK: Resource temporarily unavailable
2023-02-15 01:12:56.063 [info] <0.443.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta10" in 2000 ms after error IO error: lock ./data/swc_meta/meta10/LOCK: Resource temporarily unavailable
2023-02-15 01:12:58.066 [info] <0.443.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta10" in 2000 ms after error IO error: lock ./data/swc_meta/meta10/LOCK: Resource temporarily unavailable
2023-02-15 01:13:00.071 [info] <0.443.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta10" in 2000 ms after error IO error: lock ./data/swc_meta/meta10/LOCK: Resource temporarily unavailable
2023-02-15 01:13:02.074 [info] <0.443.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta10" in 2000 ms after error IO error: lock ./data/swc_meta/meta10/LOCK: Resource temporarily unavailable
2023-02-15 01:13:04.078 [info] <0.443.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta10" in 2000 ms after error IO error: lock ./data/swc_meta/meta10/LOCK: Resource temporarily unavailable
2023-02-15 01:13:06.082 [info] <0.443.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta10" in 2000 ms after error IO error: lock ./data/swc_meta/meta10/LOCK: Resource temporarily unavailable
2023-02-15 01:13:08.085 [info] <0.443.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta10" in 2000 ms after error IO error: lock ./data/swc_meta/meta10/LOCK: Resource temporarily unavailable
2023-02-15 01:13:10.088 [info] <0.443.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta10" in 2000 ms after error IO error: lock ./data/swc_meta/meta10/LOCK: Resource temporarily unavailable
2023-02-15 01:13:12.092 [info] <0.443.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta10" in 2000 ms after error IO error: lock ./data/swc_meta/meta10/LOCK: Resource temporarily unavailable
2023-02-15 01:13:14.096 [info] <0.443.0>@vmq_swc_db_leveldb:open_db:{293,29} VerneMQ LevelDB SWC Store backend retrying "./data/swc_meta/meta10" in 2000 ms after error IO error: lock ./data/swc_meta/meta10/LOCK: Resource temporarily unavailable
2023-02-15 01:13:16.097 [error] <0.442.0> Supervisor vmq_swc_store_sup_meta10 had child {vmq_swc_db_leveldb,meta10} started with vmq_swc_db_leveldb:start_link({swc_config,{'VerneMQ@127.0.0.1',<<239,174,60,145,53,39,215,115,159,70,176,124,80,197,199,248,97,...>>},...}, []) at undefined exit with reason {db_open,"IO error: lock ./data/swc_meta/meta10/LOCK: Resource temporarily unavailable"} in context start_error
2023-02-15 01:13:16.098 [error] <0.443.0> CRASH REPORT Process <0.443.0> with 0 neighbours exited with reason: {db_open,"IO error: lock ./data/swc_meta/meta10/LOCK: Resource temporarily unavailable"} in gen_server:init_it/6 line 835
2023-02-15 01:13:16.169 [info] <0.215.0>@vmq_metadata:start:{29,5} Try to start vmq_swc: ok
2023-02-15 01:13:16.190 [info] <0.458.0>@vmq_storage_engine_leveldb:open_db:{166,29} VerneMQ LevelDB Message Store backend retrying "./data/msgstore/1" in 2000 ms after error IO error: lock ./data/msgstore/1/LOCK: Resource temporarily unavailable
2023-02-15 01:13:18.194 [info] <0.458.0>@vmq_storage_engine_leveldb:open_db:{166,29} VerneMQ LevelDB Message Store backend retrying "./data/msgstore/1" in 2000 ms after error IO error: lock ./data/msgstore/1/LOCK: Resource temporarily unavailable
2023-02-15 01:13:20.197 [info] <0.458.0>@vmq_storage_engine_leveldb:open_db:{166,29} VerneMQ LevelDB Message Store backend retrying "./data/msgstore/1" in 2000 ms after error IO error: lock ./data/msgstore/1/LOCK: Resource temporarily unavailable
2023-02-15 01:13:22.200 [info] <0.458.0>@vmq_storage_engine_leveldb:open_db:{166,29} VerneMQ LevelDB Message Store backend retrying "./data/msgstore/1" in 2000 ms after error IO error: lock ./data/msgstore/1/LOCK: Resource temporarily unavailable
2023-02-15 01:13:24.203 [info] <0.458.0>@vmq_storage_engine_leveldb:open_db:{166,29} VerneMQ LevelDB Message Store backend retrying "./data/msgstore/1" in 2000 ms after error IO error: lock ./data/msgstore/1/LOCK: Resource temporarily unavailable
2023-02-15 01:13:26.206 [info] <0.458.0>@vmq_storage_engine_leveldb:open_db:{166,29} VerneMQ LevelDB Message Store backend retrying "./data/msgstore/1" in 2000 ms after error IO error: lock ./data/msgstore/1/LOCK: Resource temporarily unavailable
2023-02-15 01:13:28.209 [info] <0.458.0>@vmq_storage_engine_leveldb:open_db:{166,29} VerneMQ LevelDB Message Store backend retrying "./data/msgstore/1" in 2000 ms after error IO error: lock ./data/msgstore/1/LOCK: Resource temporarily unavailable
2023-02-15 01:13:30.212 [info] <0.458.0>@vmq_storage_engine_leveldb:open_db:{166,29} VerneMQ LevelDB Message Store backend retrying "./data/msgstore/1" in 2000 ms after error IO error: lock ./data/msgstore/1/LOCK: Resource temporarily unavailable
2023-02-15 01:13:32.215 [info] <0.458.0>@vmq_storage_engine_leveldb:open_db:{166,29} VerneMQ LevelDB Message Store backend retrying "./data/msgstore/1" in 2000 ms after error IO error: lock ./data/msgstore/1/LOCK: Resource temporarily unavailable
2023-02-15 01:13:34.218 [info] <0.458.0>@vmq_storage_engine_leveldb:open_db:{166,29} VerneMQ LevelDB Message Store backend retrying "./data/msgstore/1" in 2000 ms after error IO error: lock ./data/msgstore/1/LOCK: Resource temporarily unavailable
2023-02-15 01:13:36.221 [info] <0.458.0>@vmq_storage_engine_leveldb:open_db:{166,29} VerneMQ LevelDB Message Store backend retrying "./data/msgstore/1" in 2000 ms after error IO error: lock ./data/msgstore/1/LOCK: Resource temporarily unavailable
2023-02-15 01:13:38.224 [info] <0.458.0>@vmq_storage_engine_leveldb:open_db:{166,29} VerneMQ LevelDB Message Store backend retrying "./data/msgstore/1" in 2000 ms after error IO error: lock ./data/msgstore/1/LOCK: Resource temporarily unavailable
2023-02-15 01:13:40.227 [info] <0.458.0>@vmq_storage_engine_leveldb:open_db:{166,29} VerneMQ LevelDB Message Store backend retrying "./data/msgstore/1" in 2000 ms after error IO error: lock ./data/msgstore/1/LOCK: Resource temporarily unavailable
2023-02-15 01:13:42.230 [info] <0.458.0>@vmq_storage_engine_leveldb:open_db:{166,29} VerneMQ LevelDB Message Store backend retrying "./data/msgstore/1" in 2000 ms after error IO error: lock ./data/msgstore/1/LOCK: Resource temporarily unavailable
2023-02-15 01:13:44.233 [info] <0.458.0>@vmq_storage_engine_leveldb:open_db:{166,29} VerneMQ LevelDB Message Store backend retrying "./data/msgstore/1" in 2000 ms after error IO error: lock ./data/msgstore/1/LOCK: Resource temporarily unavailable
2023-02-15 01:13:46.238 [info] <0.458.0>@vmq_storage_engine_leveldb:open_db:{166,29} VerneMQ LevelDB Message Store backend retrying "./data/msgstore/1" in 2000 ms after error IO error: lock ./data/msgstore/1/LOCK: Resource temporarily unavailable
2023-02-15 01:13:48.241 [info] <0.458.0>@vmq_storage_engine_leveldb:open_db:{166,29} VerneMQ LevelDB Message Store backend retrying "./data/msgstore/1" in 2000 ms after error IO error: lock ./data/msgstore/1/LOCK: Resource temporarily unavailable
2023-02-15 01:13:50.245 [info] <0.458.0>@vmq_storage_engine_leveldb:open_db:{166,29} VerneMQ LevelDB Message Store backend retrying "./data/msgstore/1" in 2000 ms after error IO error: lock ./data/msgstore/1/LOCK: Resource temporarily unavailable
2023-02-15 01:13:52.249 [info] <0.458.0>@vmq_storage_engine_leveldb:open_db:{166,29} VerneMQ LevelDB Message Store backend retrying "./data/msgstore/1" in 2000 ms after error IO error: lock ./data/msgstore/1/LOCK: Resource temporarily unavailable
2023-02-15 01:13:54.252 [info] <0.458.0>@vmq_storage_engine_leveldb:open_db:{166,29} VerneMQ LevelDB Message Store backend retrying "./data/msgstore/1" in 2000 ms after error IO error: lock ./data/msgstore/1/LOCK: Resource temporarily unavailable
2023-02-15 01:13:56.254 [info] <0.458.0>@vmq_storage_engine_leveldb:open_db:{166,29} VerneMQ LevelDB Message Store backend retrying "./data/msgstore/1" in 2000 ms after error IO error: lock ./data/msgstore/1/LOCK: Resource temporarily unavailable
2023-02-15 01:13:58.258 [info] <0.458.0>@vmq_storage_engine_leveldb:open_db:{166,29} VerneMQ LevelDB Message Store backend retrying "./data/msgstore/1" in 2000 ms after error IO error: lock ./data/msgstore/1/LOCK: Resource temporarily unavailable
2023-02-15 01:14:00.262 [info] <0.458.0>@vmq_storage_engine_leveldb:open_db:{166,29} VerneMQ LevelDB Message Store backend retrying "./data/msgstore/1" in 2000 ms after error IO error: lock ./data/msgstore/1/LOCK: Resource temporarily unavailable
2023-02-15 01:14:02.265 [info] <0.458.0>@vmq_storage_engine_leveldb:open_db:{166,29} VerneMQ LevelDB Message Store backend retrying "./data/msgstore/1" in 2000 ms after error IO error: lock ./data/msgstore/1/LOCK: Resource temporarily unavailable
2023-02-15 01:14:04.269 [info] <0.458.0>@vmq_storage_engine_leveldb:open_db:{166,29} VerneMQ LevelDB Message Store backend retrying "./data/msgstore/1" in 2000 ms after error IO error: lock ./data/msgstore/1/LOCK: Resource temporarily unavailable
2023-02-15 01:14:06.272 [info] <0.458.0>@vmq_storage_engine_leveldb:open_db:{166,29} VerneMQ LevelDB Message Store backend retrying "./data/msgstore/1" in 2000 ms after error IO error: lock ./data/msgstore/1/LOCK: Resource temporarily unavailable
2023-02-15 01:14:08.275 [info] <0.458.0>@vmq_storage_engine_leveldb:open_db:{166,29} VerneMQ LevelDB Message Store backend retrying "./data/msgstore/1" in 2000 ms after error IO error: lock ./data/msgstore/1/LOCK: Resource temporarily unavailable
2023-02-15 01:14:10.279 [info] <0.458.0>@vmq_storage_engine_leveldb:open_db:{166,29} VerneMQ LevelDB Message Store backend retrying "./data/msgstore/1" in 2000 ms after error IO error: lock ./data/msgstore/1/LOCK: Resource temporarily unavailable
2023-02-15 01:14:12.284 [info] <0.458.0>@vmq_storage_engine_leveldb:open_db:{166,29} VerneMQ LevelDB Message Store backend retrying "./data/msgstore/1" in 2000 ms after error IO error: lock ./data/msgstore/1/LOCK: Resource temporarily unavailable
2023-02-15 01:14:14.286 [info] <0.458.0>@vmq_storage_engine_leveldb:open_db:{166,29} VerneMQ LevelDB Message Store backend retrying "./data/msgstore/1" in 2000 ms after error IO error: lock ./data/msgstore/1/LOCK: Resource temporarily unavailable
2023-02-15 01:14:16.288 [error] <0.458.0> CRASH REPORT Process <0.458.0> with 0 neighbours exited with reason: {db_open,"IO error: lock ./data/msgstore/1/LOCK: Resource temporarily unavailable"} in gen_server:init_it/6 line 835
2023-02-15 01:14:16.289 [error] <0.455.0>@vmq_generic_msg_store_sup:-start_link/0-lc$^0/1-0-:43 CRASH REPORT Process <0.455.0> with 0 neighbours exited with reason: no match of right hand value {error,{{db_open,"IO error: lock ./data/msgstore/1/LOCK: Resource temporarily unavailable"},{child,undefined,{vmq_generic_msg_store_bucket,1},{vmq_generic_msg_store,start_link,[1]},permanent,false,5000,worker,[vmq_generic_msg_store]}}} in vmq_generic_msg_store_sup:'-start_link/0-lc$^0/1-0-'/2 line 43 in application_master:init/4 line 142
2023-02-15 01:14:16.289 [info] <0.44.0>@vmq_generic_msg_store_sup:-start_link/0-lc$^0/1-0-:43 Application vmq_generic_msg_store exited with reason: no match of right hand value {error,{{db_open,"IO error: lock ./data/msgstore/1/LOCK: Resource temporarily unavailable"},{child,undefined,{vmq_generic_msg_store_bucket,1},{vmq_generic_msg_store,start_link,[1]},permanent,false,5000,worker,[vmq_generic_msg_store]}}} in vmq_generic_msg_store_sup:'-start_link/0-lc$^0/1-0-'/2 line 43
2023-02-15 01:14:16.299 [error] <0.208.0>@vmq_plugin_mgr:start_plugin:573 gen_server vmq_plugin_mgr terminated with reason: no match of right hand value {error,{vmq_generic_msg_store,{bad_return,{{vmq_generic_msg_store_app,start,[normal,[]]},{'EXIT',{{badmatch,{error,{{db_open,"IO error: lock ./data/msgstore/1/LOCK: Resource temporarily unavailable"},{child,undefined,{vmq_generic_msg_store_bucket,1},{vmq_generic_msg_store,start_link,[1]},permanent,false,5000,worker,[vmq_generic_msg_store]}}}},[{vmq_generic_msg_store_sup,'-start_link/0-lc$^0/1-0-',2,[{file,"/Users/tejarammurthykarri/Desktop/vernemq/apps/vmq_generic_msg_store/src/vmq_generic..."},...]},...]}}}}}} in vmq_plugin_mgr:start_plugin/1 line 573
2023-02-15 01:14:16.300 [error] <0.208.0>@vmq_plugin_mgr:start_plugin:573 CRASH REPORT Process vmq_plugin_mgr with 1 neighbours crashed with reason: no match of right hand value {error,{vmq_generic_msg_store,{bad_return,{{vmq_generic_msg_store_app,start,[normal,[]]},{'EXIT',{{badmatch,{error,{{db_open,"IO error: lock ./data/msgstore/1/LOCK: Resource temporarily unavailable"},{child,undefined,{vmq_generic_msg_store_bucket,1},{vmq_generic_msg_store,start_link,[1]},permanent,false,5000,worker,[vmq_generic_msg_store]}}}},[{vmq_generic_msg_store_sup,'-start_link/0-lc$^0/1-0-',2,[{file,"/Users/tejarammurthykarri/Desktop/vernemq/apps/vmq_generic_msg_store/src/vmq_generic..."},...]},...]}}}}}} in vmq_plugin_mgr:start_plugin/1 line 573
2023-02-15 01:14:16.300 [error] <0.207.0>@vmq_plugin_mgr:start_plugin:573 Supervisor vmq_plugin_sup had child vmq_plugin_mgr started with vmq_plugin_mgr:start_link() at <0.208.0> exit with reason no match of right hand value {error,{vmq_generic_msg_store,{bad_return,{{vmq_generic_msg_store_app,start,[normal,[]]},{'EXIT',{{badmatch,{error,{{db_open,"IO error: lock ./data/msgstore/1/LOCK: Resource temporarily unavailable"},{child,undefined,{vmq_generic_msg_store_bucket,1},{vmq_generic_msg_store,start_link,[1]},permanent,false,5000,worker,[vmq_generic_msg_store]}}}},[{vmq_generic_msg_store_sup,'-start_link/0-lc$^0/1-0-',2,[{file,"/Users/tejarammurthykarri/Desktop/vernemq/apps/vmq_generic_msg_store/src/vmq_generic..."},...]},...]}}}}}} in vmq_plugin_mgr:start_plugin/1 line 573 in context child_terminated
2023-02-15 01:14:16.300 [error] <0.214.0> CRASH REPORT Process <0.214.0> with 0 neighbours exited with reason: {{{badmatch,{error,{vmq_generic_msg_store,{bad_return,{{vmq_generic_msg_store_app,start,[normal,[]]},{'EXIT',{{badmatch,{error,{{db_open,"IO error: lock ./data/msgstore/1/LOCK: Resource temporarily unavailable"},{child,undefined,{vmq_generic_msg_store_bucket,1},{vmq_generic_msg_store,start_link,[1]},permanent,false,5000,worker,[vmq_generic_msg_store]}}}},[{vmq_generic_msg_store_sup,'-start_link/0-lc$^0/1-0-',2,[{file,"/Users/tejarammurthykarri/Desktop/vernemq/apps/vmq_generic_msg_store/s..."},...]},...]}}}}}}},...},...} in application_master:init/4 line 142
2023-02-15 01:14:16.301 [info] <0.44.0> Application vmq_server exited with reason: {{{badmatch,{error,{vmq_generic_msg_store,{bad_return,{{vmq_generic_msg_store_app,start,[normal,[]]},{'EXIT',{{badmatch,{error,{{db_open,"IO error: lock ./data/msgstore/1/LOCK: Resource temporarily unavailable"},{child,undefined,{vmq_generic_msg_store_bucket,1},{vmq_generic_msg_store,start_link,[1]},permanent,false,5000,worker,[vmq_generic_msg_store]}}}},[{vmq_generic_msg_store_sup,'-start_link/0-lc$^0/1-0-',2,[{file,"/Users/tejarammurthykarri/Desktop/vernemq/apps/vmq_generic_msg_store/s..."},...]},...]}}}}}}},...},...}
2023-02-15 01:14:16.304 [info] <0.89.0> alarm_handler: {clear,system_memory_high_watermark}
2023-02-15 02:03:32.308 [warning] <0.658.0>@vmq_mqtt5_fsm:connected:{819,13} client {[],<<"anon-E+bRtHxLxf3aQUkTa2lYW7gjjgM=">>} with username undefined stopped due to keepalive expired
2023-02-15 02:03:32.309 [warning] <0.658.0>@vmq_ranch:teardown:{176,13} session stopped abnormally due to 'keep_alive_timeout'
2023-02-15 02:51:09.761 [warning] <0.639.0>@vmq_mqtt5_fsm:connected:{819,13} client {[],<<"anon-4x4ebpDKTyStWLqGGyvZVhA4V+0=">>} with username undefined stopped due to keepalive expired
2023-02-15 02:51:09.761 [warning] <0.639.0>@vmq_ranch:teardown:{176,13} session stopped abnormally due to 'keep_alive_timeout'
2023-02-16 20:20:12.710 [info] <0.66.0> SIGTERM received - shutting down
2023-02-16 20:20:12.783 [info] <0.89.0> alarm_handler: {clear,system_memory_high_watermark}
2023-02-16 21:16:11.552 [info] <0.89.0> alarm_handler: {set,{system_memory_high_watermark,[]}}
2023-02-16 21:16:11.586 [info] <0.222.0>@vmq_swc_peer_service_manager:write_old_actor_to_disk:{180,13} writing (updated) old actor <<239,174,60,145,53,39,215,115,159,70,176,124,80,197,199,248,97,198,2,60>> to disk
2023-02-16 21:16:11.589 [info] <0.222.0>@vmq_swc_peer_service_manager:write_state_to_disk:{163,13} writing state {[{[{actor,<<239,174,60,145,53,39,215,115,159,70,176,124,80,197,199,248,97,198,2,60>>}],1},{<<239,174,60,145,53,39,215,115,159,70,176,124,80,197,199,248,97,198,2,60>>,1}],{dict,1,16,16,8,80,48,{[],[],[],[],[],[],[],[],[],[],[],[],[],[],[],[]},{{[],[],[['VerneMQ1@127.0.0.2',{<<239,174,60,145,53,39,215,115,159,70,176,124,80,197,199,248,97,198,2,60>>,1}]],[],[],[],[],[],[],[],[],[],[],[],[],[]}}},{dict,0,16,16,8,80,48,{[],[],[],[],[],[],[],[],[],[],[],[],[],[],[],[]},{{[],[],[],[],[],[],[],[],[],[],[],[],[],[],[],[]}}}} to disk <<75,2,131,80,0,0,1,5,120,1,203,96,206,97,96,96,96,202,96,2,81,140,25,76,41,12,172,137,201,37,249,69,185,64,174,200,251,117,54,19,77,213,175,23,207,119,219,80,19,112,244,248,143,196,99,76,54,89,137,64,85,56,101,19,25,179,50,56,83,24,88,82,50,147,75,18,25,19,5,128,144,35,49,32,209,32,67,32,11,13,100,48,130,196,192,182,130,93,144,194,32,20,150,90,148,151,234,27,104,232,96,104,100,174,103,0,132,70,248,109,66,53,17,97,47,3,97,123,81,181,2,0,216,251,86,133>>
2023-02-16 21:16:11.664 [info] <0.228.0>@vmq_swc_db_leveldb:open_db:{269,13} Opening LevelDB SWC database at "./data/swc_meta/meta1"
2023-02-16 21:16:11.678 [info] <0.238.0>@vmq_swc_db_leveldb:open_db:{269,13} Opening LevelDB SWC database at "./data/swc_meta/meta2"
2023-02-16 21:16:11.682 [info] <0.247.0>@vmq_swc_db_leveldb:open_db:{269,13} Opening LevelDB SWC database at "./data/swc_meta/meta3"
2023-02-16 21:16:11.686 [info] <0.256.0>@vmq_swc_db_leveldb:open_db:{269,13} Opening LevelDB SWC database at "./data/swc_meta/meta4"
2023-02-16 21:16:11.691 [info] <0.265.0>@vmq_swc_db_leveldb:open_db:{269,13} Opening LevelDB SWC database at "./data/swc_meta/meta5"
2023-02-16 21:16:11.695 [info] <0.274.0>@vmq_swc_db_leveldb:open_db:{269,13} Opening LevelDB SWC database at "./data/swc_meta/meta6"
2023-02-16 21:16:11.699 [info] <0.283.0>@vmq_swc_db_leveldb:open_db:{269,13} Opening LevelDB SWC database at "./data/swc_meta/meta7"
2023-02-16 21:16:11.704 [info] <0.292.0>@vmq_swc_db_leveldb:open_db:{269,13} Opening LevelDB SWC database at "./data/swc_meta/meta8"
2023-02-16 21:16:11.709 [info] <0.301.0>@vmq_swc_db_leveldb:open_db:{269,13} Opening LevelDB SWC database at "./data/swc_meta/meta9"
2023-02-16 21:16:11.713 [info] <0.310.0>@vmq_swc_db_leveldb:open_db:{269,13} Opening LevelDB SWC database at "./data/swc_meta/meta10"
2023-02-16 21:16:11.734 [info] <0.215.0>@vmq_metadata:start:{29,5} Try to start vmq_swc: ok
2023-02-16 21:16:11.749 [info] <0.323.0>@vmq_storage_engine_leveldb:open_db:{145,13} Opening LevelDB database at "./data/msgstore/1"
2023-02-16 21:16:11.751 [info] <0.324.0>@vmq_storage_engine_leveldb:open_db:{145,13} Opening LevelDB database at "./data/msgstore/2"
2023-02-16 21:16:11.753 [info] <0.325.0>@vmq_storage_engine_leveldb:open_db:{145,13} Opening LevelDB database at "./data/msgstore/3"
2023-02-16 21:16:11.754 [info] <0.326.0>@vmq_storage_engine_leveldb:open_db:{145,13} Opening LevelDB database at "./data/msgstore/4"
2023-02-16 21:16:11.756 [info] <0.327.0>@vmq_storage_engine_leveldb:open_db:{145,13} Opening LevelDB database at "./data/msgstore/5"
2023-02-16 21:16:11.758 [info] <0.328.0>@vmq_storage_engine_leveldb:open_db:{145,13} Opening LevelDB database at "./data/msgstore/6"
2023-02-16 21:16:11.759 [info] <0.329.0>@vmq_storage_engine_leveldb:open_db:{145,13} Opening LevelDB database at "./data/msgstore/7"
2023-02-16 21:16:11.761 [info] <0.330.0>@vmq_storage_engine_leveldb:open_db:{145,13} Opening LevelDB database at "./data/msgstore/8"
2023-02-16 21:16:11.762 [info] <0.331.0>@vmq_storage_engine_leveldb:open_db:{145,13} Opening LevelDB database at "./data/msgstore/9"
2023-02-16 21:16:11.764 [info] <0.332.0>@vmq_storage_engine_leveldb:open_db:{145,13} Opening LevelDB database at "./data/msgstore/10"
2023-02-16 21:16:11.765 [info] <0.333.0>@vmq_storage_engine_leveldb:open_db:{145,13} Opening LevelDB database at "./data/msgstore/11"
2023-02-16 21:16:11.767 [info] <0.334.0>@vmq_storage_engine_leveldb:open_db:{145,13} Opening LevelDB database at "./data/msgstore/12"
2023-02-16 21:16:11.784 [info] <0.215.0>@vmq_message_store:start:{29,5} Try to start vmq_generic_msg_store: ok
2023-02-16 21:16:11.940 [info] <0.336.0>@vmq_server_sup:maybe_change_nodename:{62,13} rename VerneMQ node from 'VerneMQ1@127.0.0.2' to 'VerneMQ@127.0.0.1'
2023-02-16 21:16:11.972 [info] <0.225.0>@vmq_swc_peer_service_manager:write_state_to_disk:{163,13} writing state {[{[{actor,<<239,174,60,145,53,39,215,115,159,70,176,124,80,197,199,248,97,198,2,60>>}],1},{<<239,174,60,145,53,39,215,115,159,70,176,124,80,197,199,248,97,198,2,60>>,2}],{dict,1,16,16,8,80,48,{[],[],[],[],[],[],[],[],[],[],[],[],[],[],[],[]},{{[],[['VerneMQ@127.0.0.1',{<<239,174,60,145,53,39,215,115,159,70,176,124,80,197,199,248,97,198,2,60>>,2}]],[],[],[],[],[],[],[],[],[],[],[],[],[],[]}}},{dict,0,16,16,8,80,48,{[],[],[],[],[],[],[],[],[],[],[],[],[],[],[],[]},{{[],[],[],[],[],[],[],[],[],[],[],[],[],[],[],[]}}}} to disk <<75,2,131,80,0,0,1,4,120,1,203,96,206,97,96,96,96,202,96,2,81,140,25,76,41,12,172,137,201,37,249,69,185,64,174,200,251,117,54,19,77,213,175,23,207,119,219,80,19,112,244,248,143,196,99,76,54,89,137,64,85,56,101,19,153,178,50,56,83,24,88,82,50,147,75,18,25,19,5,128,144,35,49,32,209,32,67,32,11,13,100,48,2,197,192,150,130,29,144,194,32,24,150,90,148,151,234,27,232,96,104,100,174,103,0,132,134,248,237,65,55,15,110,45,3,65,107,209,180,2,0,141,231,86,84>>
2023-02-16 21:16:12.045 [info] <0.444.0>@vmq_reg_trie:handle_info:{254,5} loaded 0 subscriptions into vmq_reg_trie
2023-02-16 21:16:12.048 [info] <0.226.0>@vmq_cluster:init:{163,5} cluster event handler 'vmq_cluster' registered
2023-02-16 21:16:12.062 [error] <0.460.0>@vmq_ranch_config:reconfigure_listeners_for_type:{269,13} can't reconfigure mqtts listener({127,0,0,1}, 1883) with Options [{max_connections,10000},{nr_of_acceptors,10},{mountpoint,[]},{depth,1},{eccs,[sect571r1,sect571k1,secp521r1,brainpoolP512r1,sect409k1,sect409r1,brainpoolP384r1,secp384r1,sect283k1,sect283r1,brainpoolP256r1,secp256k1,secp256r1,sect239k1,sect233k1,sect233r1,secp224k1,secp224r1,sect193r1,sect193r2,secp192k1,secp192r1,sect163k1,sect163r1,sect163r2,secp160k1,secp160r1,secp160r2]},{require_certificate,false},{tls_version,'tlsv1.2'},{use_identity_as_username,false},{allowed_protocol_versions,[3,4,131]},{allow_anonymous_override,false}] due to {already_started,<0.461.0>}
2023-02-16 21:16:27.388 [error] <0.208.0>@vmq_plugin_mgr:start_plugin:576 gen_server vmq_plugin_mgr terminated with reason: no match of right hand value {error,{elixir,{"no such file or directory","elixir.app"}}} in vmq_plugin_mgr:start_plugin/1 line 576
2023-02-16 21:16:27.388 [error] <0.208.0>@vmq_plugin_mgr:start_plugin:576 CRASH REPORT Process vmq_plugin_mgr with 0 neighbours crashed with reason: no match of right hand value {error,{elixir,{"no such file or directory","elixir.app"}}} in vmq_plugin_mgr:start_plugin/1 line 576
2023-02-16 21:16:27.388 [error] <0.207.0>@vmq_plugin_mgr:start_plugin:576 Supervisor vmq_plugin_sup had child vmq_plugin_mgr started with vmq_plugin_mgr:start_link() at <0.208.0> exit with reason no match of right hand value {error,{elixir,{"no such file or directory","elixir.app"}}} in vmq_plugin_mgr:start_plugin/1 line 576 in context child_terminated
2023-02-16 21:17:29.537 [error] <0.604.0>@vmq_plugin_mgr:start_plugin:576 gen_server vmq_plugin_mgr terminated with reason: no match of right hand value {error,{elixir,{"no such file or directory","elixir.app"}}} in vmq_plugin_mgr:start_plugin/1 line 576
2023-02-16 21:17:29.537 [error] <0.604.0>@vmq_plugin_mgr:start_plugin:576 CRASH REPORT Process vmq_plugin_mgr with 0 neighbours crashed with reason: no match of right hand value {error,{elixir,{"no such file or directory","elixir.app"}}} in vmq_plugin_mgr:start_plugin/1 line 576
2023-02-16 21:17:29.537 [error] <0.207.0>@vmq_plugin_mgr:start_plugin:576 Supervisor vmq_plugin_sup had child vmq_plugin_mgr started with vmq_plugin_mgr:start_link() at <0.604.0> exit with reason no match of right hand value {error,{elixir,{"no such file or directory","elixir.app"}}} in vmq_plugin_mgr:start_plugin/1 line 576 in context child_terminated
2023-02-16 21:20:37.462 [error] <0.617.0>@vmq_plugin_mgr:start_plugin:576 gen_server vmq_plugin_mgr terminated with reason: no match of right hand value {error,{elixir,{"no such file or directory","elixir.app"}}} in vmq_plugin_mgr:start_plugin/1 line 576
2023-02-16 21:20:37.462 [error] <0.617.0>@vmq_plugin_mgr:start_plugin:576 CRASH REPORT Process vmq_plugin_mgr with 0 neighbours crashed with reason: no match of right hand value {error,{elixir,{"no such file or directory","elixir.app"}}} in vmq_plugin_mgr:start_plugin/1 line 576
2023-02-16 21:20:37.462 [error] <0.207.0>@vmq_plugin_mgr:start_plugin:576 Supervisor vmq_plugin_sup had child vmq_plugin_mgr started with vmq_plugin_mgr:start_link() at <0.617.0> exit with reason no match of right hand value {error,{elixir,{"no such file or directory","elixir.app"}}} in vmq_plugin_mgr:start_plugin/1 line 576 in context child_terminated
2023-02-16 21:29:57.348 [error] <0.642.0>@vmq_plugin_mgr:start_plugin:576 gen_server vmq_plugin_mgr terminated with reason: no match of right hand value {error,{elixir,{"no such file or directory","elixir.app"}}} in vmq_plugin_mgr:start_plugin/1 line 576
2023-02-16 21:29:57.348 [error] <0.642.0>@vmq_plugin_mgr:start_plugin:576 CRASH REPORT Process vmq_plugin_mgr with 0 neighbours crashed with reason: no match of right hand value {error,{elixir,{"no such file or directory","elixir.app"}}} in vmq_plugin_mgr:start_plugin/1 line 576
2023-02-16 21:29:57.348 [error] <0.207.0>@vmq_plugin_mgr:start_plugin:576 Supervisor vmq_plugin_sup had child vmq_plugin_mgr started with vmq_plugin_mgr:start_link() at <0.642.0> exit with reason no match of right hand value {error,{elixir,{"no such file or directory","elixir.app"}}} in vmq_plugin_mgr:start_plugin/1 line 576 in context child_terminated
2023-02-16 21:34:51.785 [error] <0.706.0>@vmq_plugin_mgr:start_plugin:576 gen_server vmq_plugin_mgr terminated with reason: no match of right hand value {error,{elixir,{"no such file or directory","elixir.app"}}} in vmq_plugin_mgr:start_plugin/1 line 576
2023-02-16 21:34:51.785 [error] <0.706.0>@vmq_plugin_mgr:start_plugin:576 CRASH REPORT Process vmq_plugin_mgr with 0 neighbours crashed with reason: no match of right hand value {error,{elixir,{"no such file or directory","elixir.app"}}} in vmq_plugin_mgr:start_plugin/1 line 576
2023-02-16 21:34:51.785 [error] <0.207.0>@vmq_plugin_mgr:start_plugin:576 Supervisor vmq_plugin_sup had child vmq_plugin_mgr started with vmq_plugin_mgr:start_link() at <0.706.0> exit with reason no match of right hand value {error,{elixir,{"no such file or directory","elixir.app"}}} in vmq_plugin_mgr:start_plugin/1 line 576 in context child_terminated
2023-02-16 21:40:58.178 [error] <0.745.0>@vmq_plugin_mgr:start_plugin:576 gen_server vmq_plugin_mgr terminated with reason: no match of right hand value {error,{elixir,{"no such file or directory","elixir.app"}}} in vmq_plugin_mgr:start_plugin/1 line 576
2023-02-16 21:40:58.178 [error] <0.745.0>@vmq_plugin_mgr:start_plugin:576 CRASH REPORT Process vmq_plugin_mgr with 0 neighbours crashed with reason: no match of right hand value {error,{elixir,{"no such file or directory","elixir.app"}}} in vmq_plugin_mgr:start_plugin/1 line 576
2023-02-16 21:40:58.178 [error] <0.207.0>@vmq_plugin_mgr:start_plugin:576 Supervisor vmq_plugin_sup had child vmq_plugin_mgr started with vmq_plugin_mgr:start_link() at <0.745.0> exit with reason no match of right hand value {error,{elixir,{"no such file or directory","elixir.app"}}} in vmq_plugin_mgr:start_plugin/1 line 576 in context child_terminated
2023-02-16 21:56:32.632 [error] <0.789.0>@vmq_plugin_mgr:start_plugin:576 gen_server vmq_plugin_mgr terminated with reason: no match of right hand value {error,{elixir,{"no such file or directory","elixir.app"}}} in vmq_plugin_mgr:start_plugin/1 line 576
2023-02-16 21:56:32.632 [error] <0.789.0>@vmq_plugin_mgr:start_plugin:576 CRASH REPORT Process vmq_plugin_mgr with 0 neighbours crashed with reason: no match of right hand value {error,{elixir,{"no such file or directory","elixir.app"}}} in vmq_plugin_mgr:start_plugin/1 line 576
2023-02-16 21:56:32.632 [error] <0.207.0>@vmq_plugin_mgr:start_plugin:576 Supervisor vmq_plugin_sup had child vmq_plugin_mgr started with vmq_plugin_mgr:start_link() at <0.789.0> exit with reason no match of right hand value {error,{elixir,{"no such file or directory","elixir.app"}}} in vmq_plugin_mgr:start_plugin/1 line 576 in context child_terminated
2023-02-16 21:56:35.278 [error] <0.898.0>@vmq_plugin_mgr:start_plugin:576 gen_server vmq_plugin_mgr terminated with reason: no match of right hand value {error,{elixir,{"no such file or directory","elixir.app"}}} in vmq_plugin_mgr:start_plugin/1 line 576
2023-02-16 21:56:35.278 [error] <0.898.0>@vmq_plugin_mgr:start_plugin:576 CRASH REPORT Process vmq_plugin_mgr with 0 neighbours crashed with reason: no match of right hand value {error,{elixir,{"no such file or directory","elixir.app"}}} in vmq_plugin_mgr:start_plugin/1 line 576
2023-02-16 21:56:35.278 [error] <0.207.0>@vmq_plugin_mgr:start_plugin:576 Supervisor vmq_plugin_sup had child vmq_plugin_mgr started with vmq_plugin_mgr:start_link() at <0.898.0> exit with reason no match of right hand value {error,{elixir,{"no such file or directory","elixir.app"}}} in vmq_plugin_mgr:start_plugin/1 line 576 in context child_terminated
2023-02-16 22:20:32.432 [warning] <0.457.0>@vmq_cluster:recheck:{61,13} error during cluster checkup due to {error,no_matching_hook_found}
2023-02-16 22:20:35.429 [info] <0.44.0> Application logger started on node 'VerneMQ@127.0.0.1'
2023-02-16 22:20:35.429 [info] <0.44.0> Application httpoison started on node 'VerneMQ@127.0.0.1'
2023-02-16 22:20:35.430 [info] <0.44.0> Application exjsx started on node 'VerneMQ@127.0.0.1'
2023-02-16 22:20:35.430 [info] <0.44.0> Application my_vernemq_plugin started on node 'VerneMQ@127.0.0.1'
2023-02-16 22:20:42.433 [warning] <0.457.0>@vmq_cluster:recheck:{61,13} error during cluster checkup due to {error,no_matching_hook_found}
2023-02-16 22:20:52.434 [warning] <0.457.0>@vmq_cluster:recheck:{61,13} error during cluster checkup due to {error,no_matching_hook_found}
2023-02-16 22:21:02.435 [warning] <0.457.0>@vmq_cluster:recheck:{61,13} error during cluster checkup due to {error,no_matching_hook_found}
2023-02-16 22:21:12.436 [warning] <0.457.0>@vmq_cluster:recheck:{61,13} error during cluster checkup due to {error,no_matching_hook_found}
2023-02-16 22:21:22.437 [warning] <0.457.0>@vmq_cluster:recheck:{61,13} error during cluster checkup due to {error,no_matching_hook_found}
2023-02-16 22:21:32.438 [warning] <0.457.0>@vmq_cluster:recheck:{61,13} error during cluster checkup due to {error,no_matching_hook_found}
2023-02-16 22:21:42.439 [warning] <0.457.0>@vmq_cluster:recheck:{61,13} error during cluster checkup due to {error,no_matching_hook_found}
2023-02-16 22:21:52.440 [warning] <0.457.0>@vmq_cluster:recheck:{61,13} error during cluster checkup due to {error,no_matching_hook_found}
2023-02-16 22:22:02.441 [warning] <0.457.0>@vmq_cluster:recheck:{61,13} error during cluster checkup due to {error,no_matching_hook_found}
2023-02-16 22:22:12.442 [warning] <0.457.0>@vmq_cluster:recheck:{61,13} error during cluster checkup due to {error,no_matching_hook_found}
2023-02-16 22:22:22.444 [warning] <0.457.0>@vmq_cluster:recheck:{61,13} error during cluster checkup due to {error,no_matching_hook_found}
2023-02-16 22:22:32.445 [warning] <0.457.0>@vmq_cluster:recheck:{61,13} error during cluster checkup due to {error,no_matching_hook_found}
2023-02-16 22:22:42.446 [warning] <0.457.0>@vmq_cluster:recheck:{61,13} error during cluster checkup due to {error,no_matching_hook_found}
2023-02-16 22:22:52.447 [warning] <0.457.0>@vmq_cluster:recheck:{61,13} error during cluster checkup due to {error,no_matching_hook_found}
2023-02-16 22:23:02.448 [warning] <0.457.0>@vmq_cluster:recheck:{61,13} error during cluster checkup due to {error,no_matching_hook_found}
2023-02-16 22:23:12.449 [warning] <0.457.0>@vmq_cluster:recheck:{61,13} error during cluster checkup due to {error,no_matching_hook_found}
2023-02-16 22:23:22.450 [warning] <0.457.0>@vmq_cluster:recheck:{61,13} error during cluster checkup due to {error,no_matching_hook_found}
2023-02-16 22:23:32.451 [warning] <0.457.0>@vmq_cluster:recheck:{61,13} error during cluster checkup due to {error,no_matching_hook_found}
2023-02-16 22:23:42.452 [warning] <0.457.0>@vmq_cluster:recheck:{61,13} error during cluster checkup due to {error,no_matching_hook_found}
2023-02-16 22:23:52.453 [warning] <0.457.0>@vmq_cluster:recheck:{61,13} error during cluster checkup due to {error,no_matching_hook_found}
2023-02-16 22:24:02.454 [warning] <0.457.0>@vmq_cluster:recheck:{61,13} error during cluster checkup due to {error,no_matching_hook_found}
2023-02-16 22:24:12.455 [warning] <0.457.0>@vmq_cluster:recheck:{61,13} error during cluster checkup due to {error,no_matching_hook_found}
2023-02-16 22:24:22.456 [warning] <0.457.0>@vmq_cluster:recheck:{61,13} error during cluster checkup due to {error,no_matching_hook_found}
2023-02-16 22:24:32.457 [warning] <0.457.0>@vmq_cluster:recheck:{61,13} error during cluster checkup due to {error,no_matching_hook_found}
2023-02-16 22:24:42.458 [warning] <0.457.0>@vmq_cluster:recheck:{61,13} error during cluster checkup due to {error,no_matching_hook_found}
2023-02-16 22:24:52.459 [warning] <0.457.0>@vmq_cluster:recheck:{61,13} error during cluster checkup due to {error,no_matching_hook_found}
2023-02-16 22:25:02.460 [warning] <0.457.0>@vmq_cluster:recheck:{61,13} error during cluster checkup due to {error,no_matching_hook_found}
2023-02-16 22:32:30.180 [warning] <0.457.0>@vmq_cluster:recheck:{61,13} error during cluster checkup due to {error,no_matching_hook_found}
2023-02-16 22:32:40.187 [warning] <0.457.0>@vmq_cluster:recheck:{61,13} error during cluster checkup due to {error,no_matching_hook_found}
2023-02-16 22:47:09.607 [warning] <0.457.0>@vmq_cluster:recheck:{61,13} error during cluster checkup due to {error,no_matching_hook_found}
2023-02-16 22:47:19.616 [warning] <0.457.0>@vmq_cluster:recheck:{61,13} error during cluster checkup due to {error,no_matching_hook_found}
2023-02-16 22:47:55.554 [warning] <0.457.0>@vmq_cluster:recheck:{61,13} error during cluster checkup due to {error,no_matching_hook_found}
2023-02-16 23:04:02.427 [warning] <0.457.0>@vmq_cluster:recheck:{61,13} error during cluster checkup due to {error,no_matching_hook_found}
2023-02-16 23:10:26.814 [warning] <0.457.0>@vmq_cluster:recheck:{61,13} error during cluster checkup due to {error,no_matching_hook_found}
2023-02-16 23:11:57.956 [warning] <0.457.0>@vmq_cluster:recheck:{61,13} error during cluster checkup due to {error,no_matching_hook_found}
2023-02-16 23:12:07.861 [warning] <0.457.0>@vmq_cluster:recheck:{61,13} error during cluster checkup due to {error,no_matching_hook_found}
2023-02-16 23:12:24.286 [warning] <0.457.0>@vmq_cluster:recheck:{61,13} error during cluster checkup due to {error,no_matching_hook_found}
2023-02-16 23:14:05.787 [warning] <0.457.0>@vmq_cluster:recheck:{61,13} error during cluster checkup due to {error,no_matching_hook_found}
2023-02-16 23:21:38.371 [warning] <0.457.0>@vmq_cluster:recheck:{61,13} error during cluster checkup due to {error,no_matching_hook_found}
2023-02-16 23:21:48.271 [warning] <0.457.0>@vmq_cluster:recheck:{61,13} error during cluster checkup due to {error,no_matching_hook_found}
2023-02-16 23:21:58.172 [warning] <0.457.0>@vmq_cluster:recheck:{61,13} error during cluster checkup due to {error,no_matching_hook_found}
2023-02-16 23:22:08.075 [warning] <0.457.0>@vmq_cluster:recheck:{61,13} error during cluster checkup due to {error,no_matching_hook_found}
2023-02-16 23:22:17.979 [warning] <0.457.0>@vmq_cluster:recheck:{61,13} error during cluster checkup due to {error,no_matching_hook_found}
2023-02-16 23:22:27.883 [warning] <0.457.0>@vmq_cluster:recheck:{61,13} error during cluster checkup due to {error,no_matching_hook_found}
2023-02-16 23:22:37.787 [warning] <0.457.0>@vmq_cluster:recheck:{61,13} error during cluster checkup due to {error,no_matching_hook_found}
2023-02-16 23:22:47.693 [warning] <0.457.0>@vmq_cluster:recheck:{61,13} error during cluster checkup due to {error,no_matching_hook_found}
2023-02-16 23:22:57.596 [warning] <0.457.0>@vmq_cluster:recheck:{61,13} error during cluster checkup due to {error,no_matching_hook_found}
2023-02-16 23:23:07.501 [warning] <0.457.0>@vmq_cluster:recheck:{61,13} error during cluster checkup due to {error,no_matching_hook_found}
2023-02-16 23:23:17.405 [warning] <0.457.0>@vmq_cluster:recheck:{61,13} error during cluster checkup due to {error,no_matching_hook_found}
2023-02-16 23:23:27.309 [warning] <0.457.0>@vmq_cluster:recheck:{61,13} error during cluster checkup due to {error,no_matching_hook_found}
2023-02-16 23:23:31.304 [error] <0.417.0>@vmq_config:get_env:111 gen_server vmq_retain_srv terminated with reason: no case clause matching {error,no_matching_hook_found} in vmq_config:get_env/4 line 111
2023-02-16 23:23:31.304 [error] <0.417.0>@vmq_config:get_env:111 CRASH REPORT Process vmq_retain_srv with 0 neighbours exited with reason: no case clause matching {error,no_matching_hook_found} in vmq_config:get_env/4 line 111 in gen_server2:terminate/3 line 1143
2023-02-16 23:23:31.304 [error] <0.410.0>@vmq_config:get_env:111 Supervisor vmq_reg_sup had child vmq_retain_srv started with vmq_retain_srv:start_link() at <0.417.0> exit with reason no case clause matching {error,no_matching_hook_found} in vmq_config:get_env/4 line 111 in context child_terminated
2023-02-16 23:23:31.304 [error] <0.410.0>@vmq_config:get_env:111 Supervisor vmq_reg_sup had child vmq_retain_srv started with vmq_retain_srv:start_link() at <0.417.0> exit with reason no case clause matching {error,no_matching_hook_found} in vmq_config:get_env/4 line 111 in context start_error
2023-02-16 23:23:31.304 [error] <0.1146.0>@vmq_config:get_env:111 CRASH REPORT Process <0.1146.0> with 0 neighbours exited with reason: no case clause matching {error,no_matching_hook_found} in vmq_config:get_env/4 line 111 in gen_server2:init_it/6 line 586
2023-02-16 23:23:31.304 [error] <0.410.0>@vmq_config:get_env:111 Supervisor vmq_reg_sup had child vmq_retain_srv started with vmq_retain_srv:start_link() at {restarting,<0.417.0>} exit with reason no case clause matching {error,no_matching_hook_found} in vmq_config:get_env/4 line 111 in context start_error
2023-02-16 23:23:31.305 [error] <0.1147.0>@vmq_config:get_env:111 CRASH REPORT Process <0.1147.0> with 0 neighbours exited with reason: no case clause matching {error,no_matching_hook_found} in vmq_config:get_env/4 line 111 in gen_server2:init_it/6 line 586
2023-02-16 23:23:31.305 [error] <0.410.0>@vmq_config:get_env:111 Supervisor vmq_reg_sup had child vmq_retain_srv started with vmq_retain_srv:start_link() at {restarting,<0.417.0>} exit with reason no case clause matching {error,no_matching_hook_found} in vmq_config:get_env/4 line 111 in context start_error
2023-02-16 23:23:31.305 [error] <0.1148.0>@vmq_config:get_env:111 CRASH REPORT Process <0.1148.0> with 0 neighbours exited with reason: no case clause matching {error,no_matching_hook_found} in vmq_config:get_env/4 line 111 in gen_server2:init_it/6 line 586
2023-02-16 23:23:31.305 [error] <0.410.0>@vmq_config:get_env:111 Supervisor vmq_reg_sup had child vmq_retain_srv started with vmq_retain_srv:start_link() at {restarting,<0.417.0>} exit with reason no case clause matching {error,no_matching_hook_found} in vmq_config:get_env/4 line 111 in context start_error
2023-02-16 23:23:31.305 [error] <0.1149.0>@vmq_config:get_env:111 CRASH REPORT Process <0.1149.0> with 0 neighbours exited with reason: no case clause matching {error,no_matching_hook_found} in vmq_config:get_env/4 line 111 in gen_server2:init_it/6 line 586
2023-02-16 23:23:31.305 [error] <0.410.0>@vmq_config:get_env:111 Supervisor vmq_reg_sup had child vmq_retain_srv started with vmq_retain_srv:start_link() at {restarting,<0.417.0>} exit with reason no case clause matching {error,no_matching_hook_found} in vmq_config:get_env/4 line 111 in context start_error
2023-02-16 23:23:31.305 [error] <0.410.0> Supervisor vmq_reg_sup had child vmq_retain_srv started with vmq_retain_srv:start_link() at {restarting,<0.417.0>} exit with reason reached_max_restart_intensity in context shutdown
2023-02-16 23:23:31.305 [error] <0.1150.0>@vmq_config:get_env:111 CRASH REPORT Process <0.1150.0> with 0 neighbours exited with reason: no case clause matching {error,no_matching_hook_found} in vmq_config:get_env/4 line 111 in gen_server2:init_it/6 line 586
2023-02-16 23:23:31.305 [error] <0.336.0> Supervisor vmq_server_sup had child vmq_reg_sup started with vmq_reg_sup:start_link() at <0.410.0> exit with reason shutdown in context child_terminated
2023-02-16 23:23:31.306 [error] <0.1151.0>@vmq_config:get_env:111 Supervisor vmq_reg_sup had child vmq_retain_srv started with vmq_retain_srv:start_link() at undefined exit with reason no case clause matching {error,no_matching_hook_found} in vmq_config:get_env/4 line 111 in context start_error
2023-02-16 23:23:31.306 [error] <0.1154.0>@vmq_config:get_env:111 CRASH REPORT Process <0.1154.0> with 0 neighbours exited with reason: no case clause matching {error,no_matching_hook_found} in vmq_config:get_env/4 line 111 in gen_server2:init_it/6 line 586
2023-02-16 23:23:31.307 [error] <0.336.0> Supervisor vmq_server_sup had child vmq_reg_sup started with vmq_reg_sup:start_link() at <0.410.0> exit with reason {'EXIT',{{badmatch,{error,{shutdown,{failed_to_start_child,vmq_retain_srv,{{case_clause,{error,no_matching_hook_found}},[{vmq_config,get_env,4,[{file,"/Users/tejarammurthykarri/Desktop/vernemq/apps/vmq_server/src/vmq_config.erl"},{line,111}]},{vmq_retain_srv,init,1,[{file,"/Users/tejarammurthykarri/Desktop/vernemq/apps/vmq_server/src/vmq_retain_srv.erl"},{line,168}]},{gen_server2,init_it,6,[{file,"/Users/tejarammurthykarri/Desktop/vernemq/_build/default/lib/gen_server2/src/gen_server2.erl"},...]},...]}}}}},...}} in context start_error
2023-02-16 23:23:31.307 [error] <0.1155.0>@vmq_config:get_env:111 Supervisor vmq_reg_sup had child vmq_retain_srv started with vmq_retain_srv:start_link() at undefined exit with reason no case clause matching {error,no_matching_hook_found} in vmq_config:get_env/4 line 111 in context start_error
2023-02-16 23:23:31.307 [error] <0.1158.0>@vmq_config:get_env:111 CRASH REPORT Process <0.1158.0> with 0 neighbours exited with reason: no case clause matching {error,no_matching_hook_found} in vmq_config:get_env/4 line 111 in gen_server2:init_it/6 line 586
2023-02-16 23:23:31.307 [error] <0.336.0> Supervisor vmq_server_sup had child vmq_reg_sup started with vmq_reg_sup:start_link() at {restarting,<0.410.0>} exit with reason {'EXIT',{{badmatch,{error,{shutdown,{failed_to_start_child,vmq_retain_srv,{{case_clause,{error,no_matching_hook_found}},[{vmq_config,get_env,4,[{file,"/Users/tejarammurthykarri/Desktop/vernemq/apps/vmq_server/src/vmq_config.erl"},{line,111}]},{vmq_retain_srv,init,1,[{file,"/Users/tejarammurthykarri/Desktop/vernemq/apps/vmq_server/src/vmq_retain_srv.erl"},{line,168}]},{gen_server2,init_it,6,[{file,"/Users/tejarammurthykarri/Desktop/vernemq/_build/default/lib/gen_server2/src/gen_server2.erl"},...]},...]}}}}},...}} in context start_error
2023-02-16 23:23:31.307 [error] <0.1159.0>@vmq_config:get_env:111 Supervisor vmq_reg_sup had child vmq_retain_srv started with vmq_retain_srv:start_link() at undefined exit with reason no case clause matching {error,no_matching_hook_found} in vmq_config:get_env/4 line 111 in context start_error
2023-02-16 23:23:31.307 [error] <0.1162.0>@vmq_config:get_env:111 CRASH REPORT Process <0.1162.0> with 0 neighbours exited with reason: no case clause matching {error,no_matching_hook_found} in vmq_config:get_env/4 line 111 in gen_server2:init_it/6 line 586
2023-02-16 23:23:31.308 [error] <0.336.0> Supervisor vmq_server_sup had child vmq_reg_sup started with vmq_reg_sup:start_link() at {restarting,<0.410.0>} exit with reason {'EXIT',{{badmatch,{error,{shutdown,{failed_to_start_child,vmq_retain_srv,{{case_clause,{error,no_matching_hook_found}},[{vmq_config,get_env,4,[{file,"/Users/tejarammurthykarri/Desktop/vernemq/apps/vmq_server/src/vmq_config.erl"},{line,111}]},{vmq_retain_srv,init,1,[{file,"/Users/tejarammurthykarri/Desktop/vernemq/apps/vmq_server/src/vmq_retain_srv.erl"},{line,168}]},{gen_server2,init_it,6,[{file,"/Users/tejarammurthykarri/Desktop/vernemq/_build/default/lib/gen_server2/src/gen_server2.erl"},...]},...]}}}}},...}} in context start_error
2023-02-16 23:23:31.308 [error] <0.1163.0>@vmq_config:get_env:111 Supervisor vmq_reg_sup had child vmq_retain_srv started with vmq_retain_srv:start_link() at undefined exit with reason no case clause matching {error,no_matching_hook_found} in vmq_config:get_env/4 line 111 in context start_error
2023-02-16 23:23:31.308 [error] <0.1166.0>@vmq_config:get_env:111 CRASH REPORT Process <0.1166.0> with 0 neighbours exited with reason: no case clause matching {error,no_matching_hook_found} in vmq_config:get_env/4 line 111 in gen_server2:init_it/6 line 586
2023-02-16 23:23:31.311 [info] <0.1172.0>@vmq_metadata:stop:{45,9} Try to stop vmq_swc: {error,plugin_not_found}
2023-02-16 23:23:31.311 [info] <0.1171.0>@vmq_message_store:stop:{45,9} Try to stop vmq_generic_msg_store: {error,plugin_not_found}
2023-02-16 23:23:31.342 [error] <0.336.0> Supervisor vmq_server_sup had child vmq_reg_sup started with vmq_reg_sup:start_link() at {restarting,<0.410.0>} exit with reason {'EXIT',{{badmatch,{error,{shutdown,{failed_to_start_child,vmq_retain_srv,{{case_clause,{error,no_matching_hook_found}},[{vmq_config,get_env,4,[{file,"/Users/tejarammurthykarri/Desktop/vernemq/apps/vmq_server/src/vmq_config.erl"},{line,111}]},{vmq_retain_srv,init,1,[{file,"/Users/tejarammurthykarri/Desktop/vernemq/apps/vmq_server/src/vmq_retain_srv.erl"},{line,168}]},{gen_server2,init_it,6,[{file,"/Users/tejarammurthykarri/Desktop/vernemq/_build/default/lib/gen_server2/src/gen_server2.erl"},...]},...]}}}}},...}} in context start_error
2023-02-16 23:23:31.343 [error] <0.1167.0>@vmq_config:get_env:111 Supervisor vmq_reg_sup had child vmq_retain_srv started with vmq_retain_srv:start_link() at undefined exit with reason no case clause matching {error,no_matching_hook_found} in vmq_config:get_env/4 line 111 in context start_error
2023-02-16 23:23:31.343 [error] <0.1170.0>@vmq_config:get_env:111 CRASH REPORT Process <0.1170.0> with 0 neighbours exited with reason: no case clause matching {error,no_matching_hook_found} in vmq_config:get_env/4 line 111 in gen_server2:init_it/6 line 586
2023-02-16 23:23:31.343 [error] <0.336.0> Supervisor vmq_server_sup had child vmq_reg_sup started with vmq_reg_sup:start_link() at {restarting,<0.410.0>} exit with reason {'EXIT',{{badmatch,{error,{shutdown,{failed_to_start_child,vmq_retain_srv,{{case_clause,{error,no_matching_hook_found}},[{vmq_config,get_env,4,[{file,"/Users/tejarammurthykarri/Desktop/vernemq/apps/vmq_server/src/vmq_config.erl"},{line,111}]},{vmq_retain_srv,init,1,[{file,"/Users/tejarammurthykarri/Desktop/vernemq/apps/vmq_server/src/vmq_retain_srv.erl"},{line,168}]},{gen_server2,init_it,6,[{file,"/Users/tejarammurthykarri/Desktop/vernemq/_build/default/lib/gen_server2/src/gen_server2.erl"},...]},...]}}}}},...}} in context start_error
2023-02-16 23:23:31.343 [error] <0.336.0> Supervisor vmq_server_sup had child vmq_reg_sup started with vmq_reg_sup:start_link() at {restarting,<0.410.0>} exit with reason reached_max_restart_intensity in context shutdown
2023-02-16 23:23:31.343 [info] <0.44.0> Application vmq_server exited with reason: shutdown
2023-02-16 23:23:31.343 [info] <0.89.0> alarm_handler: {clear,system_memory_high_watermark}
2023-02-16 23:23:57.434 [info] <0.89.0> alarm_handler: {set,{system_memory_high_watermark,[]}}
2023-02-16 23:23:57.493 [info] <0.222.0>@vmq_swc_peer_service_manager:write_old_actor_to_disk:{180,13} writing (updated) old actor <<239,174,60,145,53,39,215,115,159,70,176,124,80,197,199,248,97,198,2,60>> to disk
2023-02-16 23:23:57.497 [info] <0.222.0>@vmq_swc_peer_service_manager:write_state_to_disk:{163,13} writing state {[{[{actor,<<239,174,60,145,53,39,215,115,159,70,176,124,80,197,199,248,97,198,2,60>>}],1},{<<239,174,60,145,53,39,215,115,159,70,176,124,80,197,199,248,97,198,2,60>>,2}],{dict,1,16,16,8,80,48,{[],[],[],[],[],[],[],[],[],[],[],[],[],[],[],[]},{{[],[['VerneMQ@127.0.0.1',{<<239,174,60,145,53,39,215,115,159,70,176,124,80,197,199,248,97,198,2,60>>,2}]],[],[],[],[],[],[],[],[],[],[],[],[],[],[]}}},{dict,0,16,16,8,80,48,{[],[],[],[],[],[],[],[],[],[],[],[],[],[],[],[]},{{[],[],[],[],[],[],[],[],[],[],[],[],[],[],[],[]}}}} to disk <<75,2,131,80,0,0,1,4,120,1,203,96,206,97,96,96,96,202,96,2,81,140,25,76,41,12,172,137,201,37,249,69,185,64,174,200,251,117,54,19,77,213,175,23,207,119,219,80,19,112,244,248,143,196,99,76,54,89,137,64,85,56,101,19,153,178,50,56,83,24,88,82,50,147,75,18,25,19,5,128,144,35,49,32,209,32,67,32,11,13,100,48,2,197,192,150,130,29,144,194,32,24,150,90,148,151,234,27,232,96,104,100,174,103,0,132,134,248,237,65,55,15,110,45,3,65,107,209,180,2,0,141,231,86,84>>
2023-02-16 23:23:57.520 [info] <0.228.0>@vmq_swc_db_leveldb:open_db:{269,13} Opening LevelDB SWC database at "./data/swc_meta/meta1"
2023-02-16 23:23:57.532 [info] <0.238.0>@vmq_swc_db_leveldb:open_db:{269,13} Opening LevelDB SWC database at "./data/swc_meta/meta2"
2023-02-16 23:23:57.536 [info] <0.247.0>@vmq_swc_db_leveldb:open_db:{269,13} Opening LevelDB SWC database at "./data/swc_meta/meta3"
2023-02-16 23:23:57.538 [info] <0.256.0>@vmq_swc_db_leveldb:open_db:{269,13} Opening LevelDB SWC database at "./data/swc_meta/meta4"
2023-02-16 23:23:57.544 [info] <0.265.0>@vmq_swc_db_leveldb:open_db:{269,13} Opening LevelDB SWC database at "./data/swc_meta/meta5"
2023-02-16 23:23:57.547 [info] <0.274.0>@vmq_swc_db_leveldb:open_db:{269,13} Opening LevelDB SWC database at "./data/swc_meta/meta6"
2023-02-16 23:23:57.551 [info] <0.283.0>@vmq_swc_db_leveldb:open_db:{269,13} Opening LevelDB SWC database at "./data/swc_meta/meta7"
2023-02-16 23:23:57.553 [info] <0.292.0>@vmq_swc_db_leveldb:open_db:{269,13} Opening LevelDB SWC database at "./data/swc_meta/meta8"
2023-02-16 23:23:57.557 [info] <0.301.0>@vmq_swc_db_leveldb:open_db:{269,13} Opening LevelDB SWC database at "./data/swc_meta/meta9"
2023-02-16 23:23:57.561 [info] <0.310.0>@vmq_swc_db_leveldb:open_db:{269,13} Opening LevelDB SWC database at "./data/swc_meta/meta10"
2023-02-16 23:23:57.687 [info] <0.215.0>@vmq_metadata:start:{29,5} Try to start vmq_swc: ok
2023-02-16 23:23:57.703 [info] <0.323.0>@vmq_storage_engine_leveldb:open_db:{145,13} Opening LevelDB database at "./data/msgstore/1"
2023-02-16 23:23:57.705 [info] <0.324.0>@vmq_storage_engine_leveldb:open_db:{145,13} Opening LevelDB database at "./data/msgstore/2"
2023-02-16 23:23:57.706 [info] <0.325.0>@vmq_storage_engine_leveldb:open_db:{145,13} Opening LevelDB database at "./data/msgstore/3"
2023-02-16 23:23:57.707 [info] <0.326.0>@vmq_storage_engine_leveldb:open_db:{145,13} Opening LevelDB database at "./data/msgstore/4"
2023-02-16 23:23:57.708 [info] <0.327.0>@vmq_storage_engine_leveldb:open_db:{145,13} Opening LevelDB database at "./data/msgstore/5"
2023-02-16 23:23:57.710 [info] <0.328.0>@vmq_storage_engine_leveldb:open_db:{145,13} Opening LevelDB database at "./data/msgstore/6"
2023-02-16 23:23:57.711 [info] <0.329.0>@vmq_storage_engine_leveldb:open_db:{145,13} Opening LevelDB database at "./data/msgstore/7"
2023-02-16 23:23:57.713 [info] <0.330.0>@vmq_storage_engine_leveldb:open_db:{145,13} Opening LevelDB database at "./data/msgstore/8"
2023-02-16 23:23:57.714 [info] <0.331.0>@vmq_storage_engine_leveldb:open_db:{145,13} Opening LevelDB database at "./data/msgstore/9"
2023-02-16 23:23:57.716 [info] <0.332.0>@vmq_storage_engine_leveldb:open_db:{145,13} Opening LevelDB database at "./data/msgstore/10"
2023-02-16 23:23:57.717 [info] <0.333.0>@vmq_storage_engine_leveldb:open_db:{145,13} Opening LevelDB database at "./data/msgstore/11"
2023-02-16 23:23:57.718 [info] <0.334.0>@vmq_storage_engine_leveldb:open_db:{145,13} Opening LevelDB database at "./data/msgstore/12"
2023-02-16 23:23:57.739 [info] <0.215.0>@vmq_message_store:start:{29,5} Try to start vmq_generic_msg_store: ok
2023-02-16 23:23:57.942 [info] <0.430.0>@vmq_reg_trie:handle_info:{254,5} loaded 0 subscriptions into vmq_reg_trie
2023-02-16 23:23:57.944 [info] <0.226.0>@vmq_cluster:init:{163,5} cluster event handler 'vmq_cluster' registered
2023-02-16 23:23:57.957 [error] <0.446.0>@vmq_ranch_config:reconfigure_listeners_for_type:{269,13} can't reconfigure mqtts listener({127,0,0,1}, 1883) with Options [{max_connections,10000},{nr_of_acceptors,10},{mountpoint,[]},{depth,1},{eccs,[sect571r1,sect571k1,secp521r1,brainpoolP512r1,sect409k1,sect409r1,brainpoolP384r1,secp384r1,sect283k1,sect283r1,brainpoolP256r1,secp256k1,secp256r1,sect239k1,sect233k1,sect233r1,secp224k1,secp224r1,sect193r1,sect193r2,secp192k1,secp192r1,sect163k1,sect163r1,sect163r2,secp160k1,secp160r1,secp160r2]},{require_certificate,false},{tls_version,'tlsv1.2'},{use_identity_as_username,false},{allowed_protocol_versions,[3,4,131]},{allow_anonymous_override,false}] due to {already_started,<0.447.0>}
